<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.27">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Publications – SLU Bioinformatics Infrastructure</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-ed96de9b727972fe78a7b5d16c58bf87.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-0c6282e9eda64d97ea9e5255f7a37655.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN" && texText && texText.data) {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar floating nav-fixed quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./images/SLUBI_logo.png" alt="" class="navbar-logo light-content">
    <img src="./images/SLUBI_logo.png" alt="" class="navbar-logo dark-content">
    </a>
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">SLU Bioinformatics Infrastructure</span>
    </a>
  </div>
        <div class="quarto-navbar-tools tools-end">
    <div class="dropdown">
      <a href="" title="" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" role="link" aria-label=""><i class="bi bi-github"></i></a>
      <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item quarto-navbar-tools-item" href="https://github.com/SLUBioinformaticsInfrastructure/quarto_homepage">
            Source Code
            </a>
          </li>
      </ul>
    </div>
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./slubi.html">About us</a></li><li class="breadcrumb-item"><a href="./publications.html">Publications</a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Home</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Welcome to SLUBI</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Services</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./support_what.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Support</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dropin.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Drop-in</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true">
 <span class="menu-text">Training</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./courses.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Courses</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./workshops.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Seminars and Workshops</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./learning-resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Self learning</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./online-resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tools and resources</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dardel.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">How to: Dardel</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./hpc2n.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">How to: HPC2N</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true">
 <span class="menu-text">Events</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./events.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Events</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./social.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Fika</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true">
 <span class="menu-text">About us</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./slubi.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">SLUBI</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./contact.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Contact</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./board.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Board</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./publications.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">Publications</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./theses.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Theses</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true">
 <span class="menu-text">Archive</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./news.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">News and Events</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./slubi.html">About us</a></li><li class="breadcrumb-item"><a href="./publications.html">Publications</a></li></ol></nav>
<div class="quarto-title">
<h1 class="title">Publications</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>




<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<section class="level2">
<h2 class="anchored" data-anchor-id="">2026</h2>
<ol type="1">
<li><div id="ref-WOS:001607447200001" class="csl-entry" role="listitem">
Chaudhary, S., Piombo, E., Dubey, M., Jensen, D. F., Grenville-Briggs, L. and Karlsson, M. (2026). Transcriptomic analysis of two wheat genotypes in the presence of the pathogen zymoseptoria tritici and the biological control agent clonostachys rosea [Article]. <em>PHYSIOLOGICAL AND MOLECULAR PLANT PATHOLOGY</em>, <em>141</em>. <a href="https://doi.org/10.1016/j.pmpp.2025.103013">https://doi.org/10.1016/j.pmpp.2025.103013</a>
</div></li>
</ol>
</section>
<section class="level2">
<h2 class="anchored" data-anchor-id="">2025</h2>
<ol start="2" type="1">
<li><div id="ref-liu2025genetic" class="csl-entry" role="listitem">
Alemu, A., Anguluri, K., Yuan, B., Lametsch, R. and Hammenhag, C. (2025). Genetic variation in storage protein and starch biosynthesis genes reveals key associations with seed composition in pea (pisum sativum). <em>Frontiers in Plant Science</em>. <a href="https://doi.org/10.3389/fpls.2025.1679498">https://doi.org/10.3389/fpls.2025.1679498</a>
</div></li>
<li><div id="ref-Bengtsson2025_bryophytePhenology" class="csl-entry" role="listitem">
Bengtsson, F., Cronberg, N., Lozano Villegas, J. A., Siddique, A. B., Stenberg, P. and Ekroos, J. (2025). Rapid shifts in bryophyte phenology revealed by airborne eDNA. <em>Journal of Ecology</em>. <a href="https://doi.org/10.1111/1365-2745.70180">https://doi.org/10.1111/1365-2745.70180</a>
</div></li>
<li><div id="ref-WOS:001504314200002" class="csl-entry" role="listitem">
Bergholm, J., Tessema, T. S., Blomstroem, A.-L. and Berg, M. (2025). Metagenomic insights into the complex viral composition of the enteric RNA virome in healthy and diarrheic calves from ethiopia [Article]. <em>VIROLOGY JOURNAL</em>, <em>22</em>(1). <a href="https://doi.org/10.1186/s12985-025-02821-8">https://doi.org/10.1186/s12985-025-02821-8</a>
</div></li>
<li><div id="ref-WOS:001513584400001" class="csl-entry" role="listitem">
Chekole, W. S., Potgieter, L., Adamu, H., Sternberg-Lewerin, S., Tessema, T. S. and Magnusson, U. (2025). Genomic insights into antimicrobial resistance and virulence of e. Coli in central ethiopia: A one health approach [Article]. <em>FRONTIERS IN MICROBIOLOGY</em>, <em>16</em>. <a href="https://doi.org/10.3389/fmicb.2025.1597580">https://doi.org/10.3389/fmicb.2025.1597580</a>
</div></li>
<li><div id="ref-choudhary2025tracing" class="csl-entry" role="listitem">
Choudhary, S., Renström, A., Kochakarn, T., Piombo, E., Miskolczi, P., Jämtgård, S., Henriksson, J. and Tuominen, H. (2025). Tracing the dynamics of nitrate responses in xylem elements of hybrid aspen wood [Preprint]. <em>Research Square</em>. <a href="https://doi.org/10.21203/rs.3.rs-7685028/v1">https://doi.org/10.21203/rs.3.rs-7685028/v1</a>
</div></li>
<li><div id="ref-WOS:001574453100001" class="csl-entry" role="listitem">
Cojkic, A., Niazi, A., Hansson, I. and Morrell, J. M. (2025). Variations in bacterial profiles associated with semen collection timing and bull breed, analyzed using 16S rRNA sequencing and MALDI-TOF MS [Article]. <em>FRONTIERS IN VETERINARY SCIENCE</em>, <em>12</em>. <a href="https://doi.org/10.3389/fvets.2025.1583136">https://doi.org/10.3389/fvets.2025.1583136</a>
</div></li>
<li><div id="ref-derba2025glucuronoyl" class="csl-entry" role="listitem">
Derba-Maceluch, M., Garcı́a Romañach, L., Hedenström, M., Mitra, M., Donev, E. N., Urbancsok, J., Yassin, Z., Gandla, M. L., Sivan, P., Šimura, J., et al. (2025). Glucuronoyl esterase expressed in aspen xylem affects <span class="math inline">\(\gamma\)</span>-ester linkages between lignin and glucuronoxylan reducing recalcitrance and accelerating growth. <em>Plant Biotechnology Journal</em>. <a href="https://doi.org/10.1111/pbi.70301">https://doi.org/10.1111/pbi.70301</a>
</div></li>
<li><div id="ref-WOS:001529851600003" class="csl-entry" role="listitem">
Hossein Banabazi, M., Van Borm, S., Klingstrom, T., Niazi, A., De Clercq, K., Mostin, L., Haegeman, A. and Koning, D. J. de. (2025). Host transcriptome profiling for resistance against lumpy skin disease (LSD) [Article]. <em>BMC RESEARCH NOTES</em>, <em>18</em>(1). <a href="https://doi.org/10.1186/s13104-025-07388-9">https://doi.org/10.1186/s13104-025-07388-9</a>
</div></li>
<li><div id="ref-hvidsten2025evo" class="csl-entry" role="listitem">
Hvidsten, T., Rodriguez, E., Birkeland, S., Chapple, E., Fredriksson, S., Lorenzo, Z., Kalman, T., Kumar, V., Mccann, J., Hill, J., et al. (2025). An evo-devo resource for wood: Comparative regulomics across dicot and conifer trees [Preprint]. <em>Research Square</em>. <a href="https://doi.org/10.21203/rs.3.rs-7656402/v1">https://doi.org/10.21203/rs.3.rs-7656402/v1</a>
</div></li>
<li><div id="ref-WOS:001463540700004" class="csl-entry" role="listitem">
Landi, M., Carluccio, A. V., Shah, T., Niazi, A., Stavolone, L., Falquet, L., Gisel, A. and Bongcam-Rudloff, E. (2025). Genome-wide comparison reveals large structural variants in cassava landraces [Article]. <em>BMC GENOMICS</em>, <em>26</em>(1). <a href="https://doi.org/10.1186/s12864-025-11523-y">https://doi.org/10.1186/s12864-025-11523-y</a>
</div></li>
<li><div id="ref-marcon2025transcriptional" class="csl-entry" role="listitem">
Marcon, A., Romañach, L. G., André, D., Ding, J., Zhang, B., Hvidsten, T. R. and Nilsson, O. (2025). A transcriptional roadmap of the yearly growth cycle in populus trees. <em>The Plant Cell</em>, <em>37</em>(9), koaf208. p. <a href="https://doi.org/10.1093/plcell/koaf208">https://doi.org/10.1093/plcell/koaf208</a>
</div></li>
<li><div id="ref-WOS:001598704400002" class="csl-entry" role="listitem">
Puthanveed, V., Sajeevan, R. S., Siddique, A. B., Alexandersson, E., Joshi, P., Snell, P., Lennefors, B.-L. and Kvarnheden, A. (2025). Transcriptomic responses of beet to infection by beet mild yellowing virus [Article]. <em>BMC PLANT BIOLOGY</em>, <em>25</em>(1). <a href="https://doi.org/10.1186/s12870-025-07514-6">https://doi.org/10.1186/s12870-025-07514-6</a>
</div></li>
<li><div id="ref-WOS:001529773900029" class="csl-entry" role="listitem">
Quevedo, M., Kubalova, I., Brun, A., Cervela-Cardona, L., Monte, E. and Strand, A. (2025). Retrograde signals control dynamic changes to the chromatin state at photosynthesis-associated loci [Article]. <em>Nature Communications</em>, <em>16</em>(1). <a href="https://doi.org/10.1038/s41467-025-61831-w">https://doi.org/10.1038/s41467-025-61831-w</a>
</div></li>
<li><div id="ref-rodriguez2025comparative" class="csl-entry" role="listitem">
Rodriguez, E., Birkeland, S., Chapple, E. D., Fredriksson, S., Lorenzo, Z. C., Kalman, T. A., Kumar, V., Hill, J., Tuominen, H., Mellerowicz, E. J., et al. (2025). Comparative regulomics provides novel insight into the evolution of wood formation across dicot and conifer trees [Preprint]. <em>bioRxiv</em>, 2025–2006. pp.
</div></li>
<li><div id="ref-schneider2025reverse" class="csl-entry" role="listitem">
Schneider, A. N., Buckley, S., Lorenzo, Z. C., Gratz, R., Nilsson, L., Swaine, M., Street, N. R., Taylor, A. F. and Jämtgård, S. (2025). Reverse microdialysis of sucrose stimulates soil fungal and bacterial growth at the microscale. <em>BMC Microbiology</em>, <em>25</em>(1), 436. p. <a href="https://doi.org/10.1186/s12866-025-04082-5">https://doi.org/10.1186/s12866-025-04082-5</a>
</div></li>
<li><div id="ref-WOS:001553168400001" class="csl-entry" role="listitem">
Schwochow, D., Alameddine, A., Sporndly-Nees, E., Montigny, M., Naboulsi, R., Jansson, A., Niazi, A. and Lindgren, G. (2025). RNA-seq analysis identifies key genes enhancing hoof strength to withstand barefoot racing in standardbred trotters [Article]. <em>BMC GENOMICS</em>, <em>26</em>(1). <a href="https://doi.org/10.1186/s12864-025-11814-4">https://doi.org/10.1186/s12864-025-11814-4</a>
</div></li>
<li><div id="ref-WOS:001459933600043" class="csl-entry" role="listitem">
Sigurdardottir, H., Eriksson, S., Niazi, A., Rhodin, M., Albertsdottir, E., Kristjansson, T. and Lindgren, G. (2025). Genetic influence of a STAU2 frameshift mutation and RELN regulatory elements on performance in icelandic horses [Article]. <em>SCIENTIFIC REPORTS</em>, <em>15</em>(1). <a href="https://doi.org/10.1038/s41598-025-95593-8">https://doi.org/10.1038/s41598-025-95593-8</a>
</div></li>
<li><div id="ref-WOS:001337801200001" class="csl-entry" role="listitem">
Sivan, P., Urbancsok, J., Donev, E. N., Derba-Maceluch, M., Barbut, F. R., Yassin, Z., Gandla, M. L., Mitra, M., Heinonen, S. E., Simura, J., Cermanova, K., Karady, M., Scheepers, G., Jonsson, L. J., Master, E. R., Vilaplana, F. and Mellerowicz, E. J. (2025). Modification of xylan in secondary walls alters cell wall biosynthesis and wood formation programs and improves saccharification [Article]. <em>PLANT BIOTECHNOLOGY JOURNAL</em>, <em>23</em>(1), 174–197. pp. <a href="https://doi.org/10.1111/pbi.14487">https://doi.org/10.1111/pbi.14487</a>
</div></li>
<li><div id="ref-WOS:001476550600001" class="csl-entry" role="listitem">
Taliadoros, D., Potgieter, L., Dhiman, A., Wyatt, N. A., McMullan, M., Jung, C., Bolton, M. D. and Stukenbrock, E. H. (2025). Genome-wide evidence of host specialization in wild and farmland populations of the fungal leaf spot pathogen, cercospora beticola [Article]. <em>GENOME BIOLOGY AND EVOLUTION</em>, <em>17</em>(4). <a href="https://doi.org/10.1093/gbe/evaf053">https://doi.org/10.1093/gbe/evaf053</a>
</div></li>
<li><div id="ref-urbancsok2025wood" class="csl-entry" role="listitem">
Urbancsok, J., Donev, E. N., Derba-Maceluch, M., Sivan, P., Barbut, F. R., Mitra, M., Yassin, Z., Cermanová, K., Šimura, J., Karady, M., et al. (2025). Wood-specific modification of glucuronoxylan can enhance growth in populus. <em>Journal of Experimental Botany</em>, eraf364. p. <a href="https://doi.org/10.1093/jxb/eraf364">https://doi.org/10.1093/jxb/eraf364</a>
</div></li>
<li><div id="ref-vergara2025isoformmapper" class="csl-entry" role="listitem">
Vergara, A., Hernández-Verdeja, T., Ojeda-May, P., Ramirez, L., Edler, D., Rosvall, M. and Strand, A. (2025). IsoformMapper: A web application for protein-level comparison of splice variants through structural community analysis [Preprint]. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2025.03.05.641708">https://doi.org/10.1101/2025.03.05.641708</a>
</div></li>
<li><div id="ref-vittozzi2025antagonistic" class="csl-entry" role="listitem">
Vittozzi, Y., Petri, L., Chiurazzi, M. J., Lisón, P., Hernández, C. G., Majee, A., Shankar, N., Burow, M. and Wenkel, S. (2025). Antagonistic and synergistic roles of tomato AFP3 isoforms in hormonal regulation and development. <em>bioRxiv</em>, 2025–2006. pp. <a href="https://doi.org/10.1101/2025.06.18.660365">https://doi.org/10.1101/2025.06.18.660365</a>
</div></li>
<li><div id="ref-zacharaki2025convergent" class="csl-entry" role="listitem">
Zacharaki, V., Quevedo, M., Nardeli, S. M., Meena, S. K., Monte, E. and Kindgren, P. (2025). Convergent antisense transcription primes hosting genes for stress responsiveness in plants. <em>Molecular Plant</em>, <em>18</em>(11), 1920–1931. pp. <a href="https://doi.org/10.1016/j.molp.2025.10.001">https://doi.org/10.1016/j.molp.2025.10.001</a>
</div></li>
<li><div id="ref-zhang2025unraveling" class="csl-entry" role="listitem">
Zhang, Y., Choudhary, S., Renström, A., Luomaranta, M., Chantreau, M., Fleig, V., Gaboreanu, I., Grones, C., Nilsson, O., Robinson, K. M., et al. (2025). Unraveling nitrogen uptake and metabolism: Gene families, expression dynamics and functional insights in aspen (populus tremula). <em>Tree Physiology</em>, tpaf099. p. <a href="https://doi.org/10.1093/treephys/tpaf099">https://doi.org/10.1093/treephys/tpaf099</a>
</div></li>
</ol>
</section>
<section class="level2">
<h2 class="anchored" data-anchor-id="">2024</h2>
<ol start="26" type="1">
<li><div id="ref-WOS:001201376600001" class="csl-entry" role="listitem">
Abdel-Fattah, W. R., Carlsson, M., Hu, G.-Z., Singh, A., Vergara, A., Aslam, R., Ronne, H. and Bjoerklund, S. (2024). Growth-regulated co-occupancy of mediator and Lsm3 at intronic ribosomal protein genes [Article]. <em>Nucleic Acids Research</em>, <em>52</em>(11), 6220–6233. pp. <a href="https://doi.org/10.1093/nar/gkae266">https://doi.org/10.1093/nar/gkae266</a>
</div></li>
<li><div id="ref-WOS:001245755800001" class="csl-entry" role="listitem">
Ayalew, W., Wu, X., Tarekegn, G. M., Sisay Tessema, T., Naboulsi, R., Van Damme, R., Bongcam-Rudloff, E., Edea, Z., Chu, M., Enquahone, S., Liang, C. and Yan, P. (2024). Whole genome scan uncovers candidate genes related to milk production traits in barka cattle [Article]. <em>International Journal of Molecular Sciences</em>, <em>25</em>(11). <a href="https://doi.org/10.3390/ijms25116142">https://doi.org/10.3390/ijms25116142</a>
</div></li>
<li><div id="ref-WOS:001356179600002" class="csl-entry" role="listitem">
Ayalew, W., Wu, X., Tarekegn, G. M., Tessema, T. S., Chu, M., Liang, C., Naboulsi, R., Van Damme, R., Bongcam-Rudloff, E. and Ping, Y. (2024). Whole-genome sequencing of copy number variation analysis in ethiopian cattle reveals adaptations to diverse environments [Article]. <em>BMC GENOMICS</em>, <em>25</em>(1). <a href="https://doi.org/10.1186/s12864-024-10936-5">https://doi.org/10.1186/s12864-024-10936-5</a>
</div></li>
<li><div id="ref-WOS:001335885800004" class="csl-entry" role="listitem">
Ayalew, W., Xiaoyun, W., Tarekegn, G. M., Naboulsi, R., Tessema, T. S., Van Damme, R., Bongcam-Rudloff, E., Chu, M., Liang, C., Edea, Z., Enquahone, S. and Ping, Y. (2024). Whole genome sequences of 70 indigenous ethiopian cattle [Article]. <em>Scientific Data</em>, <em>11</em>(1). <a href="https://doi.org/10.1038/s41597-024-03342-9">https://doi.org/10.1038/s41597-024-03342-9</a>
</div></li>
<li><div id="ref-WOS:001182621300050" class="csl-entry" role="listitem">
Banabazi, M. H., Freimanis, G., Goatley, L. C., Netherton, C. L. and Koning, D.-J. de. (2024). The transcriptomic insight into the differential susceptibility of african swine fever in inbred pigs [Article]. <em>Scientific Reports</em>, <em>14</em>(1). <a href="https://doi.org/10.1038/s41598-024-56569-2">https://doi.org/10.1038/s41598-024-56569-2</a>
</div></li>
<li><div id="ref-WOS:001103179900001" class="csl-entry" role="listitem">
Barcala, M. E., Valk, T. van der, Chen, Z., Funda, T., Chaudhary, R., Klingberg, A., Fundova, I., Suontama, M., Hallingback, H., Bernhardsson, C., Nystedt, B., Ingvarsson, P. K., Sherwood, E., Street, N., Gyllensten, U., Nilsson, O. and Wu, H. X. (2024). Whole-genome resequencing facilitates the development of a 50K single nucleotide polymorphism genotyping array for scots pine (pinus sylvestris l.) and its transferability to other pine species [Article]. <em>Plant Journal</em>, <em>117</em>(3), 944–955. pp. <a href="https://doi.org/10.1111/tpj.16535">https://doi.org/10.1111/tpj.16535</a>
</div></li>
<li><div id="ref-bhat2024nuclear" class="csl-entry" role="listitem">
Bhat, S. S., Asgari, M., Mermet, S., Mishra, P. and Kindgren, P. (2024). The nuclear exosome subunit HEN2 acts independently of the core exosome to assist transcription in arabidopsis. <em>Plant Physiology</em>, <em>196</em>(4), 2625–2637. pp. <a href="https://doi.org/10.1093/plphys/kiae503">https://doi.org/10.1093/plphys/kiae503</a>
</div></li>
<li><div id="ref-canovi2024resource" class="csl-entry" role="listitem">
Canovi, C., Stojkovič, K., Benı́tez, A. A., Delhomme, N., Egertsdotter, U. and Street, N. R. (2024). A resource of identified and annotated lincRNAs expressed during somatic embryogenesis development in norway spruce. <em>Physiologia Plantarum</em>, <em>176</em>(5), e14537. p. <a href="https://doi.org/10.1111/ppl.14537">https://doi.org/10.1111/ppl.14537</a>
</div></li>
<li><div id="ref-WOS:001289238100001" class="csl-entry" role="listitem">
Cojkic, A., Niazi, A. and Morrell, J. M. (2024). Metagenomic identification of bull semen microbiota in different seasons [Article]. <em>Animal Reproduction Science</em>, <em>268</em>. <a href="https://doi.org/10.1016/j.anireprosci.2024.107569">https://doi.org/10.1016/j.anireprosci.2024.107569</a>
</div></li>
<li><div id="ref-WOS:001164173200035" class="csl-entry" role="listitem">
Dube, F., Delhomme, N., Martin, F., Hinas, A., Åbrink, M., Svärd, S. and Tydén, E. (2024). Gene co-expression network analysis reveal core responsive genes in parascaris univalens tissues following ivermectin exposure [Article]. <em>PLoS One</em>, <em>19</em>(2). <a href="https://doi.org/10.1371/journal.pone.0298039">https://doi.org/10.1371/journal.pone.0298039</a>
</div></li>
<li><div id="ref-el2024arabidopsis" class="csl-entry" role="listitem">
El Arbi, N., Nardeli, S. M., Šimura, J., Ljung, K. and Schmid, M. (2024). The arabidopsis splicing factor PORCUPINE/SmE1 orchestrates temperature-dependent root development via auxin homeostasis maintenance. <em>New Phytologist</em>, <em>244</em>(4), 1408–1421. pp. <a href="https://doi.org/10.1111/nph.20153">https://doi.org/10.1111/nph.20153</a>
</div></li>
<li><div id="ref-hermann2024urban" class="csl-entry" role="listitem">
Hermann, E., Van Damme, R., Bongcam-Rudloff, E. and Nasirzadeh, L. (2024). Urban pigeons as reservoirs of critical pathogens: Improved protocol for sequencing pigeon faeces in disease monitoring. <em>EMBnet. Journal</em>, <em>30</em>, 1059. p. <a href="https://doi.org/10.14806/ej.30.0.1059">https://doi.org/10.14806/ej.30.0.1059</a>
</div></li>
<li><div id="ref-hermida2024cdk8" class="csl-entry" role="listitem">
Hermida-Carrera, C., Vergara, A., Cervela-Cardona, L., Jin, X., Björklund, S. and Strand, Å. (2024). CDK8 of the mediator kinase module connects leaf development to the establishment of correct stomata patterning by regulating the levels of the transcription factor SPEECHLESS (SPCH). <em>Plant, Cell &amp; Environment</em>, <em>47</em>(12), 5237–5251. pp. <a href="https://doi.org/10.1111/pce.15102">https://doi.org/10.1111/pce.15102</a>
</div></li>
<li><div id="ref-agronomy14112682" class="csl-entry" role="listitem">
Islam, M. M., Deepo, D. M., Siddique, A. B., Nasif, S. O., Zonayet, M., Hassan, O., Siddique, A. B. and Ali, M. A. (2024). B chromosomes in wheat: Evolution, functions and breeding potential. <em>Agronomy</em>, <em>14</em>(11). <a href="https://doi.org/10.3390/agronomy14112682">https://doi.org/10.3390/agronomy14112682</a>
</div></li>
<li><div id="ref-WOS:001326514900001" class="csl-entry" role="listitem">
Jhamat, N., Guo, Y., Han, J., Humblot, P., Bongcam-Rudloff, E., Andersson, G. and Niazi, A. (2024). Enrichment of cis-acting regulatory elements in differentially methylated regions following lipopolysaccharide treatment of bovine endometrial epithelial cells [Article]. <em>International Journal of Molecular Sciences</em>, <em>25</em>(18). <a href="https://doi.org/10.3390/ijms25189832">https://doi.org/10.3390/ijms25189832</a>
</div></li>
<li><div id="ref-WOS:001363161900001" class="csl-entry" role="listitem">
Lazaro-Gimeno, D., Ferrari, C., Delhomme, N., Johansson, M., Sjolander, J., Singh, R. K., Mutwil, M. and Eriksson, M. E. (2024). The circadian clock participates in seasonal growth in norway spruce (picea abies) [Article]. <em>Tree Physiology</em>, <em>44</em>(11). <a href="https://doi.org/10.1093/treephys/tpae139">https://doi.org/10.1093/treephys/tpae139</a>
</div></li>
<li><div id="ref-WOS:001279841300001" class="csl-entry" role="listitem">
Luomaranta, M., Grones, C., Choudhary, S., Milhinhos, A., Kalman, T. A., Nilsson, O., Robinson, K. M., Street, N. R. and Tuominen, H. (2024). Systems genetic analysis of lignin biosynthesis in populus tremula [Article]. <em>New Phytologist</em>, <em>243</em>(6), 2157–2174. pp. <a href="https://doi.org/10.1111/nph.19993">https://doi.org/10.1111/nph.19993</a>
</div></li>
<li><div id="ref-WOS:001154852800002" class="csl-entry" role="listitem">
Malaluang, P., Niazi, A., Guo, Y., Nagel, C., Guimaraes, T., Rocha, A., Aurich, C. and Morrell, J. M. (2024). Bacterial diversity in semen from stallions in three european countries evaluated by 16S sequencing [Article]. <em>Veterinary Research Communications</em>, <em>48</em>(3), 1409–1421. pp. <a href="https://doi.org/10.1007/s11259-024-10321-3">https://doi.org/10.1007/s11259-024-10321-3</a>
</div></li>
<li><div id="ref-WOS:001170714200001" class="csl-entry" role="listitem">
Manourova, A., Polesny, Z., Ruiz-Chutan, A., Sillam-Dusses, D., Tsafack, S., Tchoudjeu, Z., Potgieter, L. and Lojka, B. (2024). Identification of plus trees for domestication: Phenotypical description of garcinia kola populations in cameroon. <em>Genetic Recourses and Crop Evolution</em>, <em>71</em>(8), 1893–1909. pp. <a href="https://doi.org/10.1007/s10722-023-01750-1">https://doi.org/10.1007/s10722-023-01750-1</a>
</div></li>
<li><div id="ref-meena2024antisense" class="csl-entry" role="listitem">
Meena, S. K., Quevedo, M., Nardeli, S. M., Verez, C., Bhat, S. S., Zacharaki, V. and Kindgren, P. (2024). Antisense transcription from stress-responsive transcription factors fine-tunes the cold response in arabidopsis. <em>The Plant Cell</em>, <em>36</em>(9), 3467–3482. pp. <a href="https://doi.org/10.1093/plcell/koae160">https://doi.org/10.1093/plcell/koae160</a>
</div></li>
<li><div id="ref-nardeli2024time" class="csl-entry" role="listitem">
Nardeli, S. M., El Arbi, N., Dikaya, V., Rojas-Murcia, N., Goretti, D. and Schmid, M. (2024). Time and temperature-resolved transcriptomic analysis of arabidopsis splicing-related mutants [Preprint]. <em>bioRxiv</em>, 2024–2011. pp. <a href="https://doi.org/10.1101/2024.11.08.622454">https://doi.org/10.1101/2024.11.08.622454</a>
</div></li>
<li><div id="ref-WOS:001165287900001" class="csl-entry" role="listitem">
Ohm, H., Saripella, G. V., Hofvander, P. and Grimberg, A. (2024). Spatio-temporal transcriptome and storage compound profiles of developing faba bean (vicia faba) seed tissues [Article]. <em>Frontiers in Plant Science</em>, <em>15</em>. <a href="https://doi.org/10.3389/fpls.2024.1284997">https://doi.org/10.3389/fpls.2024.1284997</a>
</div></li>
<li><div id="ref-WOS:001301300600001" class="csl-entry" role="listitem">
Resjo, S., Willforss, J., Large, A., Siino, V., Alexandersson, E., Levander, F. and Andreasson, E. (2024). Comparative proteomic analyses of potato leaves from field-grown plants grown under extremely long days [Article]. <em>Plant Physiology and Biochemistry</em>, <em>215</em>. <a href="https://doi.org/10.1016/j.plaphy.2024.109032">https://doi.org/10.1016/j.plaphy.2024.109032</a>
</div></li>
<li><div id="ref-robinson2024improved" class="csl-entry" role="listitem">
Robinson, K. M., Schiffthaler, B., Liu, H., Rydman, S. M., Rendon-Anaya, M., Kalman, T. A., Kumar, V., Canovi, C., Bernhardsson, C., Delhomme, N., Jenkins, J., Wang, J., Mahler, N., Richau, K. H., Stokes, V., A’Hara, S., Cottrell, J., Coeck, K., Diels, T., … Street, N. R. (2024a). An improved chromosome-scale genome assembly and population genetics resource for populus tremula [Article]. <em>Physiologia Plantarum</em>, <em>176</em>(5). <a href="https://doi.org/10.1111/ppl.14511">https://doi.org/10.1111/ppl.14511</a>
</div></li>
<li><div id="ref-WOS:001329224000001" class="csl-entry" role="listitem">
Robinson, K. M., Schiffthaler, B., Liu, H., Rydman, S. M., Rendon-Anaya, M., Kalman, T. A., Kumar, V., Canovi, C., Bernhardsson, C., Delhomme, N., Jenkins, J., Wang, J., Mahler, N., Richau, K. H., Stokes, V., A’Hara, S., Cottrell, J., Coeck, K., Diels, T., … Street, N. R. (2024b). Correction to an improved chromosome-scale genome assembly and population genetics resource for populus tremula [Correction]. <em>Physiologia Plantarum</em>, <em>176</em>(5). <a href="https://doi.org/10.1111/ppl.14564">https://doi.org/10.1111/ppl.14564</a>
</div></li>
<li><div id="ref-sibhat2024detection" class="csl-entry" role="listitem">
Sibhat, B., Adamu, H., Asmare, K., Lindahl, J. F., Magnusson, U. and Sisay Tessema, T. (2024). Detection and molecular diversity of brucella melitensis in pastoral livestock in north-eastern ethiopia. <em>Pathogens</em>, <em>13</em>(12), 1063. p. <a href="https://doi.org/10.3390/pathogens13121063">https://doi.org/10.3390/pathogens13121063</a>
</div></li>
<li><div id="ref-10.21203/rs.3.rs-4206868/v1" class="csl-entry" role="listitem">
Siddique, A. B., Maharwa, L. and Albrectsen, B. (2024). Genotype, tannin capacity, and seasonality influence the structure and function of symptomless fungal communities in aspen leaves, regardless of historical nitrogen addition [Preprint]. <em>Research Square</em>. <a href="https://doi.org/10.21203/rs.3.rs-4206868/v1">https://doi.org/10.21203/rs.3.rs-4206868/v1</a>
</div></li>
<li><div id="ref-spain2024transcriptomic" class="csl-entry" role="listitem">
Spain, O., Bajhaiya, A. K., Tanner, H., Cook, S. and Funk, C. (2024). Transcriptomic analysis of a cold-resistant nordic microalga: Unraveling the mechanisms underlying adaptation to low temperatures. <em>Physiologia Plantarum</em>, <em>176</em>(3), e14402. p. <a href="https://doi.org/10.1111/ppl.14402">https://doi.org/10.1111/ppl.14402</a>
</div></li>
<li><div id="ref-WOS:001342128400001" class="csl-entry" role="listitem">
Stojkovič, K., Canovi, C., Le, K.-C., Ahmad, I., Gaboreanu, I., Johansson, S., Delhomme, N., Egertsdotter, U. and Street, N. R. (2024). A transcriptome atlas of zygotic and somatic embryogenesis in norway spruce [Article]. <em>Plant Journal</em>, <em>120</em>(5), 2238–2252. pp. <a href="https://doi.org/10.1111/tpj.17087">https://doi.org/10.1111/tpj.17087</a>
</div></li>
<li><div id="ref-street2024new" class="csl-entry" role="listitem">
Street, N., Nystedt, B., Delhomme, N., Eriksson, M. C., Hill, J., Ahlgren Kalman, T., Kumar, V., Larsson, T., Nandi, S., Mähler, N., et al. (2024). New genome insights from chromosome-scale genome assemblies of norway spruce (picea abies) and scots pine (pinus sylvestris) [Preprint]. <em>Umeå Universitet, Diva</em>.
</div></li>
</ol>
</section>
<section class="level2">
<h2 class="anchored" data-anchor-id="">2023</h2>
<ol start="56" type="1">
<li><div id="ref-WOS:001093828000001" class="csl-entry" role="listitem">
Ayalew, W., Wu, X., Tarekegn, G. M., Sisay Tessema, T., Naboulsi, R., Van Damme, R., Bongcam-Rudloff, E., Edea, Z., Enquahone, S. and Yan, P. (2023). Whole-genome resequencing reveals selection signatures of abigar cattle for local adaptation [Article]. <em>ANIMALS</em>, <em>13</em>(20). <a href="https://doi.org/10.3390/ani13203269">https://doi.org/10.3390/ani13203269</a>
</div></li>
<li><div id="ref-WOS:000885655900001" class="csl-entry" role="listitem">
Bai, B., Schiffthaler, B., Horst, S. van der, Willems, L., Vergara, A., Karstrom, J., Mahler, N., Delhomme, N., Bentsink, L. and Hanson, J. (2023). SeedTransNet: A directional translational network revealing regulatory patterns during seed maturation and germination [Article]. <em>Journal of Experimental Botany</em>, <em>74</em>(7, SI), 2416–2432. pp. <a href="https://doi.org/10.1093/jxb/erac394">https://doi.org/10.1093/jxb/erac394</a>
</div></li>
<li><div id="ref-belda2023effects" class="csl-entry" role="listitem">
Belda-Perez, R., Heras, S., Cimini, C., Romero-Aguirregomezcorta, J., Valbonetti, L., Colosimo, A., Maria Colosimo, B., Santoni, S., Bernabo, N., Coy, P., et al. (2023). Effects of a polycaprolactone and polyethylene glycol diacrylate, 3-d printed scaffolds, on bovine embryo development in vitro. <em>Animal Reproduction</em>, <em>20</em>(2), 8–8. pp. <a href="https://re.public.polimi.it/retrieve/6fb345c3-60a5-4236-a703-786e4ac1551a/AETE%202023.pdf">https://re.public.polimi.it/retrieve/6fb345c3-60a5-4236-a703-786e4ac1551a/AETE%202023.pdf</a>
</div></li>
<li><div id="ref-bhat2023mta" class="csl-entry" role="listitem">
Bhat, S. S., Bielewicz, D. and Kindgren, P. (2023). MTA influences RNA polymerase II transcription dynamics and regulates the cold response in arabidopsis [Preprint]. <em>bioRxiv</em>, 2023–2005. pp. <a href="https://doi.org/10.1101/2023.05.10.540235 ">https://doi.org/10.1101/2023.05.10.540235 </a>
</div></li>
<li><div id="ref-WOS:000984919900057" class="csl-entry" role="listitem">
Dube, F., Hinas, A., Delhomme, N., abrink, M., Svard, S. and Tyden, E. (2023). Transcriptomics of ivermectin response in caenorhabditis elegans: Integrating abamectin quantitative trait loci and comparison to the ivermectin-exposed DA1316 strain [Article]. <em>PLoS One</em>, <em>18</em>(5). <a href="https://doi.org/10.1371/journal.pone.0285262">https://doi.org/10.1371/journal.pone.0285262</a>
</div></li>
<li><div id="ref-eilertsen2023microbial" class="csl-entry" role="listitem">
Eilertsen, L., Hultberg, M., Lee, N., Saroj, P., Swaine, M., Chen, F. and Xiong, S. (2023). The microbial community in a substrate of solid-state fermentation by lentinula edodes: A preliminary study. <em>Fermentation</em>, <em>9</em>(8), 736. p. <a href="https://doi.org/10.3390/fermentation9080736">https://doi.org/10.3390/fermentation9080736</a>
</div></li>
<li><div id="ref-WOS:000896714400001" class="csl-entry" role="listitem">
Galmiche, M., Sonnette, A., Wolf, M., Sutter, C., Delhomme, O., Francois, Y.-N. and Millet, M. (2023). Simultaneous determination of 79 polar and non-polar polycyclic aromatic compounds in airborne particulate matter by gas chromatography - tandem mass spectrometry [Article]. <em>Polycyclic Aromatic Compounds</em>, <em>43</em>(10), 8841–8860. pp. <a href="https://doi.org/10.1080/10406638.2022.2153884">https://doi.org/10.1080/10406638.2022.2153884</a>
</div></li>
<li><div id="ref-EJ1045" class="csl-entry" role="listitem">
Gisel, A., Stavolone, L., Olagunju, T., Landi, M., Damme, R. van, Niazi, A., Falquet, L., Shah, T. and Bongcam-Rudloff, E. (2023). EpiCass and CassavaNet4Dev advanced bioinformatics workshop. <em>EMBnet.journal</em>, <em>29</em>(0), 1045. p. <a href="https://doi.org/10.14806/ej.29.0.1045">https://doi.org/10.14806/ej.29.0.1045</a>
</div></li>
<li><div id="ref-WOS:001016857100001" class="csl-entry" role="listitem">
Hemberg, E., Niazi, A., Guo, Y., Debnar, V. J., Vincze, B., Morrell, J. M. and Kutvolgyi, G. (2023). Microbial profiling of amniotic fluid, umbilical blood and placenta of the foaling mare [Article]. <em>Animals</em>, <em>13</em>(12). <a href="https://doi.org/10.3390/ani13122029">https://doi.org/10.3390/ani13122029</a>
</div></li>
<li><div id="ref-WOS:001118804500003" class="csl-entry" role="listitem">
Landi, M., Shah, T., Falquet, L., Niazi, A., Stavolone, L., Bongcam-Rudloff, E. and Gisel, A. (2023). Haplotype-resolved genome of heterozygous african cassava cultivar TMEB117 (manihot esculenta) [Article; Data Paper]. <em>Scientific Data</em>, <em>10</em>(1). <a href="https://doi.org/10.1038/s41597-023-02800-0">https://doi.org/10.1038/s41597-023-02800-0</a>
</div></li>
<li><div id="ref-WOS:001037322100027" class="csl-entry" role="listitem">
Lihavainen, J., Simura, J., Bag, P., Fataftah, N., Robinson, K. M., Delhomme, N., Novak, O., Ljung, K. and Jansson, S. (2023). Salicylic acid metabolism and signalling coordinate senescence initiation in aspen in nature [Article]. <em>Nature Communications</em>, <em>14</em>(1). <a href="https://doi.org/10.1038/s41467-023-39564-5">https://doi.org/10.1038/s41467-023-39564-5</a>
</div></li>
<li><div id="ref-nardeli2023arabidopsis" class="csl-entry" role="listitem">
Nardeli, S. M., Zacharaki, V., Rojas-Murcia, N., Collani, S., Wang, K., Bayer, M., Schmid, M. and Goretti, D. (2023). Arabidopsis thaliana LSM7 is essential for auxin-mediated regulation of SAUR genes and thermomorphogenesis [Preprint]. <em>bioRxiv</em>, 2023–2003. pp. <a href="https://doi.org/10.1101/2023.03.28.534379 ">https://doi.org/10.1101/2023.03.28.534379 </a>
</div></li>
<li><div id="ref-WOS:001095232300001" class="csl-entry" role="listitem">
Puthanveed, V., Singh, K., Poimenopoulou, E., Pettersson, J., Siddique, A. B. and Kvarnheden, A. (2023). Milder autumns may increase risk for infection of crops with turnip yellows virus [Article]. <em>PHYTOPATHOLOGY</em>, <em>113</em>(9), 1788–1798. pp. <a href="https://doi.org/10.1094/PHYTO-11-22-0446-V">https://doi.org/10.1094/PHYTO-11-22-0446-V</a>
</div></li>
<li><div id="ref-sajeevan2023comprehensive" class="csl-entry" role="listitem">
Sajeevan, R. S., Abdelmeguid, I., Saripella, G. V., Lenman, M. and Alexandersson, E. (2023). Comprehensive transcriptome analysis of different potato cultivars provides insight into early blight disease caused by alternaria solani. <em>BMC Plant Biology</em>, <em>23</em>(1), 130. p. <a href="https://doi.org/10.1186/s12870-023-04135-9">https://doi.org/10.1186/s12870-023-04135-9</a>
</div></li>
<li><div id="ref-WOS:001021913700001" class="csl-entry" role="listitem">
Schiffthaler, B., Zalen, E. van, Serrano, A. R., Street, N. R. and Delhomme, N. (2023). Seiðr: Efficient calculation of robust ensemble gene networks [Article]. <em>HELIYON</em>, <em>9</em>(6). <a href="https://doi.org/10.1016/j.heliyon.2023.e16811">https://doi.org/10.1016/j.heliyon.2023.e16811</a>
</div></li>
<li><div id="ref-WOS:001097574200002" class="csl-entry" role="listitem">
Stenberg, H., Malmberg, M. and Hayer, J. (2023). Screening for atypical porcine pestivirus in swedish boar semen used for artificial insemination and a characterisation of the seminal RNA microbiome including the virome [Article]. <em>BMC Veterinary Research</em>, <em>19</em>(1). <a href="https://doi.org/10.1186/s12917-023-03762-6">https://doi.org/10.1186/s12917-023-03762-6</a>
</div></li>
<li><div id="ref-Sullivan2023.12.06.569882" class="csl-entry" role="listitem">
Sullivan, A. R., Karlsson, E., Svensson, D., Brindefalk, B., Villegas, J. A., Mikko, A., Bellieny, D., Siddique, A. B., Johansson, A.-M., Grahn, H., Sundell, D., Norman, A., Esseen, P.-A., Sjödin, A., Singh, N. J., Brodin, T., Forsman, M. and Stenberg, P. (2023). Airborne eDNA captures three decades of ecosystem biodiversity. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2023.12.06.569882">https://doi.org/10.1101/2023.12.06.569882</a>
</div></li>
<li><div id="ref-WOS:001085559700001" class="csl-entry" role="listitem">
Urbancsok, J., Donev, E. N., Sivan, P., Zalen, E. van, Barbut, F. R., Derba-Maceluch, M., Simura, J., Yassin, Z., Gandla, M. L., Karady, M., Ljung, K., Winestrand, S., Jonsson, L. J., Scheepers, G., Delhomme, N., Street, N. R. and Mellerowicz, E. J. (2023). Flexure wood formation via growth reprogramming in hybrid aspen involves jasmonates and polyamines and transcriptional changes resembling tension wood development [Article]. <em>New Phytologist</em>, <em>240</em>(6), 2312–2334. pp. <a href="https://doi.org/10.1111/nph.19307">https://doi.org/10.1111/nph.19307</a>
</div></li>
</ol>
</section>
<section class="level2">
<h2 class="anchored" data-anchor-id="">2022</h2>
<ol start="74" type="1">
<li><div id="ref-WOS:000854207600001" class="csl-entry" role="listitem">
Akhter, S., Westrin, K. J., Zivi, N., Nordal, V., Kretzschmar, W. W., Delhomme, N., Street, N. R., Nilsson, O., Emanuelsson, O. and Sundstrom, J. F. (2022). Cone-setting in spruce is regulated by conserved elements of the age-dependent flowering pathway [Article]. <em>New Phytologist</em>, <em>236</em>(5), 1951–1963. pp. <a href="https://doi.org/10.1111/nph.18449">https://doi.org/10.1111/nph.18449</a>
</div></li>
<li><div id="ref-andre2022flowering" class="csl-entry" role="listitem">
André, D., Marcon, A., Lee, K. C., Goretti, D., Zhang, B., Delhomme, N., Schmid, M. and Nilsson, O. (2022). FLOWERING LOCUS t paralogs control the annual growth cycle in populus trees. <em>Current Biology</em>, <em>32</em>(13), 2988–2996. pp. https://doi.org/<a href="https://doi.org/10.1016/j.cub.2022.05.023">https://doi.org/10.1016/j.cub.2022.05.023</a>
</div></li>
<li><div id="ref-BALINANDI2022198739" class="csl-entry" role="listitem">
Balinandi, S., Hayer, J., Cholleti, H., Wille, M., Lutwama, J. J., Malmberg, M. and Mugisha, L. (2022). Identification and molecular characterization of highly divergent RNA viruses in cattle, uganda. <em>Virus Research</em>, <em>313</em>, 198739. p. https://doi.org/<a href="https://doi.org/10.1016/j.virusres.2022.198739">https://doi.org/10.1016/j.virusres.2022.198739</a>
</div></li>
<li><div id="ref-WOS:000862402500001" class="csl-entry" role="listitem">
Boussardon, C., Bag, P., Juvany, M., Simura, J., Ljung, K., Jansson, S. and Keech, O. (2022). The RPN12a proteasome subunit is essential for the multiple hormonal homeostasis controlling the progression of leaf senescence [Article]. <em>Communications Biology</em>, <em>5</em>(1). <a href="https://doi.org/10.1038/s42003-022-03998-2">https://doi.org/10.1038/s42003-022-03998-2</a>
</div></li>
<li><div id="ref-WOS:000873813700001" class="csl-entry" role="listitem">
Castro, D., Concha, C., Jamett, F., Ibanez, C. and Hurry, V. (2022). Soil microbiome influences on seedling establishment and growth of prosopis chilensis and prosopis tamarugo from northern chile [Article]. <em>PLANTS-BASEL</em>, <em>11</em>(20). <a href="https://doi.org/10.3390/plants11202717">https://doi.org/10.3390/plants11202717</a>
</div></li>
<li><div id="ref-cholleti2022characterization" class="csl-entry" role="listitem">
Cholleti, H., Jong, J. de, Blomström, A.-L. and Berg, M. (2022a). Characterization of pipistrellus pygmaeus bat virome from sweden. <em>Viruses</em>, <em>14</em>(8), 1654. p. <a href="https://doi.org/10.3390/v14081654">https://doi.org/10.3390/v14081654</a>
</div></li>
<li><div id="ref-cholleti2022investigation" class="csl-entry" role="listitem">
Cholleti, H., Jong, J. de, Blomström, A.-L. and Berg, M. (2022b). Investigation of the virome and characterization of issyk-kul virus from swedish myotis brandtii bats. <em>Pathogens</em>, <em>12</em>(1), 12. p.
</div></li>
<li><div id="ref-WOS:000830354600001" class="csl-entry" role="listitem">
Chowdhury, J., Kemppainen, M., Delhomme, N., Shutava, I., Zhou, J., Takahashi, J., Pardo, A. G. and Lundberg-Felten, J. (2022). Laccaria bicolor pectin methylesterases are involved in ectomycorrhiza development with populus tremula x populus tremuloides [Article]. <em>New Phytologist</em>, <em>236</em>(2), 639–655. pp. <a href="https://doi.org/10.1111/nph.18358">https://doi.org/10.1111/nph.18358</a>
</div></li>
<li><div id="ref-dube2022ivermectin" class="csl-entry" role="listitem">
Dube, F., Hinas, A., Roy, S., Martin, F., Åbrink, M., Svärd, S. and Tydén, E. (2022). Ivermectin-induced gene expression changes in adult parascaris univalens and caenorhabditis elegans: A comparative approach to study anthelminthic metabolism and resistance in vitro. <em>Parasites &amp; Vectors</em>, <em>15</em>(1), 158. p. <a href="https://doi.org/10.1186/s13071-022-05260-4">https://doi.org/10.1186/s13071-022-05260-4</a>
</div></li>
<li><div id="ref-fegraeus2022equine" class="csl-entry" role="listitem">
Fegraeus, K., Rosengren, M. K., Naboulsi, R., Orlando, L., Åbrink, M., Thorsell, A., Jouni, A., Velie, B. D., Raine, A., Egner, B., et al. (2022). An equine endothelin 3 cis-regulatory variant links blood pressure modulation to elite racing performance [Preprint]. <em>bioRxiv</em>, 2022–2011. pp. <a href="https://doi.org/10.1101/2022.11.04.515141 ">https://doi.org/10.1101/2022.11.04.515141 </a>
</div></li>
<li><div id="ref-WOS:000794167200001" class="csl-entry" role="listitem">
Galmiche, M., Rodrigues, A., Motsch, E., Delhomme, O., Francois, Y.-N. and Millet, M. (2022). The use of pseudo-MRM for a sensitive and selective detection and quantification of polycyclic aromatic compounds by tandem mass spectrometry [Article]. <em>Rapid Communications in Mass Spectrometry</em>, <em>36</em>(13). <a href="https://doi.org/10.1002/rcm.9307">https://doi.org/10.1002/rcm.9307</a>
</div></li>
<li><div id="ref-ghadamgahi2022plant" class="csl-entry" role="listitem">
Ghadamgahi, F., Tarighi, S., Taheri, P., Saripella, G. V., Anzalone, A., Kalyandurg, P. B., Catara, V., Ortiz, R. and Vetukuri, R. R. (2022). Plant growth-promoting activity of pseudomonas aeruginosa FG106 and its ability to act as a biocontrol agent against potato, tomato and taro pathogens. <em>Biology</em>, <em>11</em>(1), 140. p. <a href="https://doi.org/10.3390/biology11010140">https://doi.org/10.3390/biology11010140</a>
</div></li>
<li><div id="ref-10.3389/fpls.2022.816425" class="csl-entry" role="listitem">
Grimberg, Å., Saripella, G. V., Repo-Carrasco Valencia, R. A.-M., Bengtsson, T., Alandia, G. and Carlsson, A. S. (2022). Transcriptional regulation of quinoa seed quality: Identification of novel candidate genetic markers for increased protein content. <em>Frontiers in Plant Science</em>, <em>Volume 13 - 2022</em>. <a href="https://doi.org/10.3389/fpls.2022.816425">https://doi.org/10.3389/fpls.2022.816425</a>
</div></li>
<li><div id="ref-hernandez2022genomes" class="csl-entry" role="listitem">
Hernández-Verdeja, T., Vuorijoki, L., Jin, X., Vergara, A., Dubreuil, C. and Strand, Å. (2022). GENOMES UNCOUPLED1 plays a key role during the de-etiolation process in arabidopsis. <em>New Phytologist</em>, <em>235</em>(1), 188–203. pp. <a href="https://doi.org/10.1111/nph.18115 ">https://doi.org/10.1111/nph.18115 </a>
</div></li>
<li><div id="ref-plants11202684" class="csl-entry" role="listitem">
Islam, M. M., Deepo, D. M., Nasif, S. O., Siddique, A. B., Hassan, O., Siddique, A. B. and Paul, N. C. (2022). Cytogenetics and consequences of polyploidization on different biotic-abiotic stress tolerance and the potential mechanisms involved. <em>Plants</em>, <em>11</em>(20). <a href="https://doi.org/10.3390/plants11202684">https://doi.org/10.3390/plants11202684</a>
</div></li>
<li><div id="ref-WOS:000900721500001" class="csl-entry" role="listitem">
Jeilu, O., Simachew, A., Alexandersson, E., Johansson, E. and Gessesse, A. (2022). Discovery of novel carbohydrate degrading enzymes from soda lakes through functional metagenomics [Article]. <em>FRONTIERS IN MICROBIOLOGY</em>, <em>13</em>. <a href="https://doi.org/10.3389/fmicb.2022.1059061">https://doi.org/10.3389/fmicb.2022.1059061</a>
</div></li>
<li><div id="ref-karamat2022arabidopsis" class="csl-entry" role="listitem">
Karamat, F., Vergara, A., Blomberg, J., Crawford, T., Lehotai, N., Rentoft, M., Strand, Å. and Björklund, S. (2022). Arabidopsis mutants representing each of the four mediator modules reveal unique functions in the transcriptional response to salt stress [Preprint]. <em>bioRxiv</em>, 2022–2006. pp. <a href="https://doi.org/10.1101/2022.06.24.497547">https://doi.org/10.1101/2022.06.24.497547</a>
</div></li>
<li><div id="ref-law2022metatranscriptomics" class="csl-entry" role="listitem">
Law, S. R., Serrano, A. R., Daguerre, Y., Sundh, J., Schneider, A. N., Stangl, Z. R., Castro, D., Grabherr, M., Näsholm, T., Street, N. R., et al. (2022). Metatranscriptomics captures dynamic shifts in mycorrhizal coordination in boreal forests. <em>Proceedings of the National Academy of Sciences</em>, <em>119</em>(26), e2118852119. p. <a href="https://doi.org/10.1073/pnas.211885211">https://doi.org/10.1073/pnas.211885211</a>
</div></li>
<li><div id="ref-liebsch2022metabolic" class="csl-entry" role="listitem">
Liebsch, D., Juvany, M., Li, Z., Wang, H.-L., Ziolkowska, A., Chrobok, D., Boussardon, C., Wen, X., Law, S. R., Janečková, H., et al. (2022). Metabolic control of arginine and ornithine levels paces the progression of leaf senescence. <em>Plant Physiology</em>, <em>189</em>(4), 1943–1960. pp. <a href="https://doi.org/10.1093/plphys/kiac244">https://doi.org/10.1093/plphys/kiac244</a>
</div></li>
<li><div id="ref-genes13122342" class="csl-entry" role="listitem">
Najafi, M., Guo, Y., Andersson, G., Humblot, P. and Bongcam-Rudloff, E. (2022). Gene networks and pathways involved in LPS-induced proliferative response of bovine endometrial epithelial cells. <em>Genes</em>, <em>13</em>(12). <a href="https://doi.org/10.3390/genes13122342">https://doi.org/10.3390/genes13122342</a>
</div></li>
<li><div id="ref-ohlund2022transcriptome" class="csl-entry" role="listitem">
Öhlund, P., Delhomme, N., Hayer, J., Hesson, J. C. and Blomström, A.-L. (2022). Transcriptome analysis of an aedes albopictus cell line single-and dual-infected with lammi virus and WNV. <em>International Journal of Molecular Sciences</em>, <em>23</em>(2), 875. p. <a href="https://doi.org/10.3390/ijms23020875">https://doi.org/10.3390/ijms23020875</a>
</div></li>
<li><div id="ref-ranade2022adaptive" class="csl-entry" role="listitem">
Ranade, S. S., Seipel, G., Gorzsas, A. and Garcı́a-Gil, M. R. (2022). Adaptive strategies of scots pine under shade: Increase in lignin synthesis and ecotypic variation in defense-related gene expression. <em>Physiologia Plantarum</em>, <em>174</em>(5), e13792. p. <a href="https://doi.org/10.1111/ppl.13792">https://doi.org/10.1111/ppl.13792</a>
</div></li>
<li><div id="ref-ranade2022enhanced" class="csl-entry" role="listitem">
Ranade, S. S., Seipel, G., Gorzsás, A. and Garcı́a-Gil, M. R. (2022). Enhanced lignin synthesis and ecotypic variation in defense-related gene expression in response to shade in norway spruce. <em>Plant, Cell &amp; Environment</em>, <em>45</em>(9), 2671–2681. pp. <a href="https://doi.org/10.1111/pce.14387">https://doi.org/10.1111/pce.14387</a>
</div></li>
<li><div id="ref-ranjan2022molecular" class="csl-entry" role="listitem">
Ranjan, A., Perrone, I., Alallaq, S., Singh, R., Rigal, A., Brunoni, F., Chitarra, W., Guinet, F., Kohler, A., Martin, F., et al. (2022). Molecular basis of differential adventitious rooting competence in poplar genotypes. <em>Journal of Experimental Botany</em>, <em>73</em>(12), 4046–4064. pp. <a href="https://doi.org/10.1093/jxb/erac126">https://doi.org/10.1093/jxb/erac126</a>
</div></li>
<li><div id="ref-siddique2022optimization" class="csl-entry" role="listitem">
Siddique, A. B., Albrectsen, B. R., Ilbi, H. and Siddique, A. B. (2022). Optimization of protocol for construction of fungal ITS amplicon library for high-throughput illumina sequencing to study the mycobiome of aspen leaves. <em>Applied Sciences</em>, <em>12</em>(3), 1136. p. <a href="https://doi.org/10.3390/app12031136">https://doi.org/10.3390/app12031136</a>
</div></li>
<li><div id="ref-WOS:000854754500001" class="csl-entry" role="listitem">
Stenberg, H., Hellman, S., Lindstrom, L., Jacobson, M., Fossum, C., Hayer, J. and Malmberg, M. (2022). Congenital tremor and splay leg in piglets - insights into the virome, local cytokine response, and histology [Article]. <em>BMC VETERINARY RESEARCH</em>, <em>18</em>(1). <a href="https://doi.org/10.1186/s12917-022-03443-w">https://doi.org/10.1186/s12917-022-03443-w</a>
</div></li>
<li><div id="ref-van2022host" class="csl-entry" role="listitem">
Van Borm, S., Klingström, T. and Niazi, A. (2022). Host variation in susceptibility to lumpy skin disease: Gene expression analyses of experimentally infected cows. <em>12th World Congress on Genetics Applied to Livestock Production (WCGALP), 2022-07-03, Wageningen, the Netherlands</em>. <a href="https://doi.org/10.3920/978-90-8686-940-4_154">https://doi.org/10.3920/978-90-8686-940-4_154</a>
</div></li>
<li><div id="ref-veerkamp2022challenges" class="csl-entry" role="listitem">
Veerkamp, R. and Haas, Y. de. (2022). Challenges–genetic control of infectious disease transmission and impact. <em>Proceedings of 12th World Congress on Genetics Applied to Livestock Production (WCGALP)</em>, 663–735. pp. <a href="https://doi.org/10.3920/9789086869404_153">https://doi.org/10.3920/9789086869404_153</a>
</div></li>
<li><div id="ref-vergara2022norway" class="csl-entry" role="listitem">
Vergara, A., Haas, J. C., Stachula, P., Street, N. R. and Hurry, V. (2022). Norway spruce deploys tissue-specific responses during acclimation to cold. <em>Plant Cell Environ</em>, <em>45</em>, 427–445. pp. <a href="https://doi.org/10.1111/pce.14241">https://doi.org/10.1111/pce.14241</a>
</div></li>
<li><div id="ref-zacharaki2022impaired" class="csl-entry" role="listitem">
Zacharaki, V., Ponnu, J., Crepin, N., Langenecker, T., Hagmann, J., Skorzinski, N., Musialak-Lange, M., Wahl, V., Rolland, F. and Schmid, M. (2022). Impaired KIN10 function restores developmental defects in the arabidopsis trehalose 6-phosphate synthase1 (tps1) mutant. <em>New Phytologist</em>, <em>235</em>(1), 220–233. pp. <a href="https://doi.org/10.1111/nph.18104">https://doi.org/10.1111/nph.18104</a>
</div></li>
</ol>
</section>
<section class="level2">
<h2 class="anchored" data-anchor-id="">2021</h2>
<ol start="104" type="1">
<li><div id="ref-azeez2021early" class="csl-entry" role="listitem">
Azeez, A., Zhao, Y. C., Singh, R. K., Yordanov, Y. S., Dash, M., Miskolczi, P., Stojkovič, K., Strauss, S. H., Bhalerao, R. P. and Busov, V. B. (2021). EARLY BUD-BREAK 1 and EARLY BUD-BREAK 3 control resumption of poplar growth after winter dormancy. <em>Nature Communications</em>, <em>12</em>(1), 1123. p. <a href="https://doi.org/10.1038/s41467-021-21449-0">https://doi.org/10.1038/s41467-021-21449-0</a>
</div></li>
<li><div id="ref-bag2021atlas" class="csl-entry" role="listitem">
Bag, P., Lihavainen, J., Delhomme, N., Riquelme, T., Robinson, K. M. and Jansson, S. (2021). An atlas of the norway spruce needle seasonal transcriptome. <em>The Plant Journal</em>, <em>108</em>(6), 1815–1829. pp. <a href="https://doi.org/10.1111/tpj.15530">https://doi.org/10.1111/tpj.15530</a>
</div></li>
<li><div id="ref-brouwer2021transcriptome" class="csl-entry" role="listitem">
Brouwer, S. M., Brus-Szkalej, M., Saripella, G. V., Liang, D., Liljeroth, E. and Grenville-Briggs, L. J. (2021). Transcriptome analysis of potato infected with the necrotrophic pathogen alternaria solani. <em>Plants</em>, <em>10</em>(10), 2212. p. <a href="https://doi.org/10.3390/plants10102212 ">https://doi.org/10.3390/plants10102212 </a>
</div></li>
<li><div id="ref-ani11113112" class="csl-entry" role="listitem">
Cabezas-Garcia, E. H., Danielsson, R., Ramin, M. and Huhtanen, P. (2021). In vitro incubations do not reflect in vivo differences based on ranking of low and high methane emitters in dairy cows. <em>Animals</em>, <em>11</em>(11). <a href="https://doi.org/10.3390/ani11113112">https://doi.org/10.3390/ani11113112</a>
</div></li>
<li><div id="ref-Christie2021_qtlXplorer" class="csl-entry" role="listitem">
Christie, N., Mannapperuma, C., Ployet, R., Merwe, K. van der, Mähler, N., Delhomme, N., Naidoo, S., Mizrachi, E., Street, N. R. and Myburg, A. A. (2021). qtlXplorer: An online systems genetics browser in the eucalyptus genome integrative explorer (EucGenIE). <em>BMC Bioinformatics</em>, <em>22</em>(1), 595. p. <a href="https://doi.org/10.1186/s12859-021-04514-9">https://doi.org/10.1186/s12859-021-04514-9</a>
</div></li>
<li><div id="ref-cojkic2021identification" class="csl-entry" role="listitem">
Cojkic, A., Niazi, A., Guo, Y., Hallap, T., Padrik, P. and Morrell, J. M. (2021). Identification of bull semen microbiome by 16S sequencing and possible relationships with fertility. <em>Microorganisms</em>, <em>9</em>(12), 2431. p. <a href="https://doi.org/10.3390/microorganisms9122431">https://doi.org/10.3390/microorganisms9122431</a>
</div></li>
<li><div id="ref-CORDUNEANU2021101605" class="csl-entry" role="listitem">
Corduneanu, A., Mihalca, A. D., Sándor, A. D., Hornok, S., Malmberg, M., Viso, N. P. and Bongcam-Rudloff, E. (2021). The heart microbiome of insectivorous bats from central and south eastern europe. <em>Comparative Immunology, Microbiology and Infectious Diseases</em>, <em>75</em>, 101605. p. https://doi.org/<a href="https://doi.org/10.1016/j.cimid.2020.101605">https://doi.org/10.1016/j.cimid.2020.101605</a>
</div></li>
<li><div id="ref-dobrenel2021phosphate" class="csl-entry" role="listitem">
Dobrenel, T., Kushwah, S., Mubeen, U., Jansen, W., Delhomme, N., Caldana, C. and Hanson, J. (2021). The phosphate starvation response recruits the TOR pathway to regulate growth in arabidopsis cell cultures. <em>bioRxiv</em>, 2021–2003. pp. <a href="https://doi.org/10.1101/2021.03.26.437164">https://doi.org/10.1101/2021.03.26.437164</a>
</div></li>
<li><div id="ref-friedrich2021sex" class="csl-entry" role="listitem">
Friedrich, A., Bardhan, A., Maekawa, H., Schacherer, J., Knop, M., Delhomme, N., Dutreux, F., Papaioannou, I. A. and Peltier, F. (2021). Sex without crossing over in the yeast saccharomycodes ludwigii. <em>Genome Biology</em>, <em>22</em>. <a href="https://doi.org/10.1186/s13059-021-02521-w">https://doi.org/10.1186/s13059-021-02521-w</a>
</div></li>
<li><div id="ref-10.1093/eurheartj/ehab030" class="csl-entry" role="listitem">
Garnier, S., Harakalova, M., Weiss, S., Mokry, M., Regitz-Zagrosek, V., Hengstenberg, C., Cappola, T. P., Isnard, R., Arbustini, E., Cook, S. A., Setten, J. van, Calis, J. J. A., Hakonarson, H., Morley, M. P., Stark, K., Prasad, S. K., Li, J., O’Regan, D. P., Grasso, M., … Charron, G. consortium, Philippe. (2021). Genome-wide association analysis in dilated cardiomyopathy reveals two new players in systolic heart failure on chromosomes 3p25.1 and 22q11.23. <em>European Heart Journal</em>, <em>42</em>(20), 2000–2011. pp. <a href="https://doi.org/10.1093/eurheartj/ehab030">https://doi.org/10.1093/eurheartj/ehab030</a>
</div></li>
<li><div id="ref-haas2021candidate" class="csl-entry" role="listitem">
Haas, J. C., Vergara, A., Serrano, A. R., Mishra, S., Hurry, V. and Street, N. R. (2021). Candidate regulators and target genes of drought stress in needles and roots of norway spruce. <em>Tree Physiology</em>, <em>41</em>(7), 1230–1246. pp. <a href="https://doi.org/10.1093/treephys/tpaa178">https://doi.org/10.1093/treephys/tpaa178</a>
</div></li>
<li><div id="ref-HAYER2021116" class="csl-entry" role="listitem">
Hayer, J., Wille, M., Font, A., González-Aravena, M., Norder, H. and Malmberg, M. (2021). Four novel picornaviruses detected in magellanic penguins (spheniscus magellanicus) in chile. <em>Virology</em>, <em>560</em>, 116–123. pp. <a href="https://doi.org/10.1016/j.virol.2021.05.010">https://doi.org/10.1016/j.virol.2021.05.010</a>
</div></li>
<li><div id="ref-Holmfeldt2021_FennoscandianShieldVirosphere" class="csl-entry" role="listitem">
Holmfeldt, K., Nilsson, E., Simone, D., Lopez-Fernandez, M., Wu, X., Bruijn, I. de, Lundin, D., Andersson, A. F., Bertilsson, S. and Dopson, M. (2021). The fennoscandian shield deep terrestrial virosphere suggests slow motion <span>“boom and burst”</span> cycles. <em>Communications Biology</em>, <em>4</em>(1), 307. p. <a href="https://doi.org/10.1038/s42003-021-01810-1">https://doi.org/10.1038/s42003-021-01810-1</a>
</div></li>
<li><div id="ref-genes12010063" class="csl-entry" role="listitem">
Juma, I., Geleta, M., Hovmalm, H. P., Nyomora, A., Saripella, G. V., Carlsson, A. S., Fatih, M. and Ortiz, R. (2021). Comparison of morphological and genetic characteristics of avocados grown in tanzania. <em>Genes</em>, <em>12</em>(1). <a href="https://doi.org/10.3390/genes12010063">https://doi.org/10.3390/genes12010063</a>
</div></li>
<li><div id="ref-KAHLERT2021108088" class="csl-entry" role="listitem">
Kahlert, M., Bailet, B., Chonova, T., Karjalainen, S. M., Schneider, S. C. and Tapolczai, K. (2021). Same same, but different: The response of diatoms to environmental gradients in fennoscandian streams and lakes – barcodes, traits and microscope data compared. <em>Ecological Indicators</em>, <em>130</em>, 108088. p. <a href="https://doi.org//10.1016/j.ecolind.2021.108088">https://doi.org//10.1016/j.ecolind.2021.108088</a>
</div></li>
<li><div id="ref-lihavainen2021stem" class="csl-entry" role="listitem">
Lihavainen, J., Edlund, E., Björkén, L., Bag, P., Robinson, K. M. and Jansson, S. (2021). Stem girdling affects the onset of autumn senescence in aspen in interaction with metabolic signals. <em>Physiologia Plantarum</em>, <em>172</em>(1), 201–217. pp. <a href="https://doi.org/10.1111/ppl.13319">https://doi.org/10.1111/ppl.13319</a>
</div></li>
<li><div id="ref-liu2021centromere" class="csl-entry" role="listitem">
Liu, H., Yan, X.-M., Wang, X., Zhang, D.-X., Zhou, Q., Shi, T.-L., Jia, K.-H., Tian, X.-C., Zhou, S.-S., Zhang, R.-G., et al. (2021). Centromere-specific retrotransposons and very-long-chain fatty acid biosynthesis in the genome of yellowhorn (xanthoceras sorbifolium, sapindaceae), an oil-producing tree with significant drought resistance. <em>Frontiers in Plant Science</em>, <em>12</em>, 766389. p. <a href="https://doi.org/10.3389/fpls.2021.766389">https://doi.org/10.3389/fpls.2021.766389</a>
</div></li>
<li><div id="ref-mahboubi2021small" class="csl-entry" role="listitem">
Mahboubi, A., Delhomme, N., Häggström, S. and Hanson, J. (2021). Small-scale sequencing enables quality assessment of ribo-seq data: An example from arabidopsis cell culture. <em>Plant Methods</em>, <em>17</em>(1), 92. p. <a href="https://doi.org/10.1186/s13007-021-00791-w">https://doi.org/10.1186/s13007-021-00791-w</a>
</div></li>
<li><div id="ref-mahmud2021prokseq" class="csl-entry" role="listitem">
Mahmud, A. F., Delhomme, N., Nandi, S. and Fällman, M. (2021). ProkSeq for complete analysis of RNA-seq data from prokaryotes. <em>Bioinformatics</em>, <em>37</em>(1), 126–128. pp. <a href="https://doi.org/10.1093/bioinformatics/btaa1063">https://doi.org/10.1093/bioinformatics/btaa1063</a>
</div></li>
<li><div id="ref-martin2021exploring" class="csl-entry" role="listitem">
Martin, F., Halvarsson, P., Delhomme, N., Höglund, J. and Tyden, E. (2021). Exploring the <span class="math inline">\(\beta\)</span>-tubulin gene family in a benzimidazole-resistant parascaris univalens population. <em>International Journal for Parasitology: Drugs and Drug Resistance</em>, <em>17</em>, 84–91. pp. <a href="https://doi.org/10.1016/j.ijpddr.2021.08.004">https://doi.org/10.1016/j.ijpddr.2021.08.004</a>
</div></li>
<li><div id="ref-Mehrshad2021_CoreMicrobiomeDeepGroundwater" class="csl-entry" role="listitem">
Mehrshad, M., Lopez-Fernandez, M., Sundh, J., Bell, E., Simone, D., Buck, M., Bernier-Latmani, R., Bertilsson, S. and Dopson, M. (2021). Energy efficiency and biological interactions define the core microbiome of deep oligotrophic groundwater. <em>Nature Communications</em>, <em>12</em>(1), 4253. p. <a href="https://doi.org/10.1038/s41467-021-24549-z">https://doi.org/10.1038/s41467-021-24549-z</a>
</div></li>
<li><div id="ref-mishra2021plastid" class="csl-entry" role="listitem">
Mishra, L. S., Mishra, S., Caddell, D. F., Coleman-Derr, D. and Funk, C. (2021). The plastid-localized AtFtsHi3 pseudo-protease of arabidopsis thaliana has an impact on plant growth and drought tolerance. <em>Frontiers in Plant Science</em>, <em>12</em>, 694727. p. <a href="https://doi.org/10.3389/fpls.2021.694727">https://doi.org/10.3389/fpls.2021.694727</a>
</div></li>
<li><div id="ref-moestedt2021production" class="csl-entry" role="listitem">
Moestedt, J., Westerholm, M., Isaksson, S. and Schnurer, A. (2021). Production during anaerobic digestion of sewage sludge and food waste. <em>Current Advances in Anaerobic Digestion Technology</em>, 153. p. <a href="https://mdpi-res.com/bookfiles/book/3502/Current_Advances_in_Anaerobic_Digestion_Technology.pdf">https://mdpi-res.com/bookfiles/book/3502/Current_Advances_in_Anaerobic_Digestion_Technology.pdf</a>
</div></li>
<li><div id="ref-morange2021rare" class="csl-entry" role="listitem">
Morange, P.-E., Peiretti, F., Gourhant, L., Proust, C., Soukarieh, O., Pulcrano-Nicolas, A.-S., Saripella, G.-V., Stefanucci, L., Lacroix, R., Ibrahim-Kosta, M., et al. (2021). A rare coding mutation in the MAST2 gene causes venous thrombosis in a french family with unexplained thrombophilia: The breizh MAST2 Arg89Gln variant. <em>PLoS Genetics</em>, <em>17</em>(1), e1009284. p. <a href="https://doi.org/10.1371/journal.pgen.1009284 ">https://doi.org/10.1371/journal.pgen.1009284 </a>
</div></li>
<li><div id="ref-ranade2021molecular" class="csl-entry" role="listitem">
Ranade, S. S. and Garcı́a-Gil, M. R. (2021). Molecular signatures of local adaptation to light in norway spruce. <em>Planta</em>, <em>253</em>(2), 53. p. <a href="https://doi.org/10.1007/s00425-020-03517-9">https://doi.org/10.1007/s00425-020-03517-9</a>
</div></li>
<li><div id="ref-schneider2021comparative" class="csl-entry" role="listitem">
Schneider, A. N., Sundh, J., Sundström, G., Richau, K., Delhomme, N., Grabherr, M., Hurry, V. and Street, N. R. (2021). Comparative fungal community analyses using metatranscriptomics and internal transcribed spacer amplicon sequencing from norway spruce. <em>Msystems</em>, <em>6</em>(1), 10–1128. pp. <a href="https://doi.org/10.1128/msystems.00884-20">https://doi.org/10.1128/msystems.00884-20</a>
</div></li>
<li><div id="ref-cells10081971" class="csl-entry" role="listitem">
Seyfferth, C., Wessels, B. A., Vahala, J., Kangasjärvi, J., Delhomme, N., Hvidsten, T. R., Tuominen, H. and Lundberg-Felten, J. (2021). Populus PtERF85 balances xylem cell expansion and secondary cell wall formation in hybrid aspen. <em>Cells</em>, <em>10</em>(8). <a href="https://doi.org/10.3390/cells10081971">https://doi.org/10.3390/cells10081971</a>
</div></li>
<li><div id="ref-VanDamme2021_MUFFIN" class="csl-entry" role="listitem">
Van Damme, R., Hölzer, M., Viehweger, A., Müller, B., Bongcam–Rudloff, E. and Brandt, C. (2021). Metagenomics workflow for hybrid assembly, differential coverage binning, metatranscriptomics and pathway analysis (MUFFIN). <em>PLOS Computational Biology</em>, <em>17</em>(2), e1008716. p. <a href="https://doi.org/10.1371/journal.pcbi.1008716">https://doi.org/10.1371/journal.pcbi.1008716</a>
</div></li>
</ol>
</section>
<section class="level2">
<h2 class="anchored" data-anchor-id="">2020</h2>
<ol start="132" type="1">
<li><div id="ref-ALKASS2020106568" class="csl-entry" role="listitem">
Al-Kass, Z., Guo, Y., Vinnere Pettersson, O., Niazi, A. and Morrell, J. M. (2020). Metagenomic analysis of bacteria in stallion semen. <em>Animal Reproduction Science</em>, <em>221</em>, 106568. p. <a href="https://doi.org/10.1016/j.anireprosci.2020.106568">https://doi.org/10.1016/j.anireprosci.2020.106568</a>
</div></li>
<li><div id="ref-brunoni2020conifers" class="csl-entry" role="listitem">
Brunoni, F., Collani, S., Casanova-Sáez, R., Šimura, J., Karady, M., Schmid, M., Ljung, K. and Bellini, C. (2020). Conifers exhibit a characteristic inactivation of auxin to maintain tissue homeostasis. <em>New Phytologist</em>, <em>226</em>(6), 1753–1765. pp. <a href="https://doi.org/10.1111/nph.16463">https://doi.org/10.1111/nph.16463</a>
</div></li>
<li><div id="ref-crawford2020specific" class="csl-entry" role="listitem">
Crawford, T., Karamat, F., Lehotai, N., Rentoft, M., Blomberg, J., Strand, Å. and Björklund, S. (2020). Specific functions for mediator complex subunits from different modules in the transcriptional response of arabidopsis thaliana to abiotic stress. <em>Scientific Reports</em>, <em>10</em>(1), 5073. p. <a href="https://doi.org/10.1038/s41598-020-61758-w">https://doi.org/10.1038/s41598-020-61758-w</a>
</div></li>
<li><div id="ref-frosth2020identification" class="csl-entry" role="listitem">
Frosth, S., Karlsson-Lindsjö, O., Niazi, A., Fernström, L.-L. and Hansson, I. (2020). Identification of transmission routes of campylobacter and on-farm measures to reduce campylobacter in chicken. <em>Pathogens</em>, <em>9</em>(5), 363. p. <a href="https://doi.org/10.3390/pathogens9050363 ">https://doi.org/10.3390/pathogens9050363 </a>
</div></li>
<li><div id="ref-gil2020transcriptomic" class="csl-entry" role="listitem">
Gil-Muñoz, F., Delhomme, N., Quiñones, A., Naval, M. del M., Badenes, M. L. and Garcı́a-Gil, M. R. (2020). Transcriptomic analysis reveals salt tolerance mechanisms present in date-plum persimmon rootstock (diospyros lotus l.). <em>Agronomy</em>, <em>10</em>(11), 1703. p. <a href="https://doi.org/10.3390/agronomy10111703">https://doi.org/10.3390/agronomy10111703</a>
</div></li>
<li><div id="ref-hammenhag2020qtl" class="csl-entry" role="listitem">
Hammenhag, C., Saripella, G. V., Ortiz, R. and Geleta, M. (2020). QTL mapping for domestication-related characteristics in field cress (lepidium campestre)—a novel oil crop for the subarctic region. <em>Genes</em>, <em>11</em>(10), 1223. p. <a href="https://10.3390/genes11101223">10.3390/genes11101223</a>
</div></li>
<li><div id="ref-jhamat2020lps" class="csl-entry" role="listitem">
Jhamat, N., Niazi, A., Guo, Y., Chanrot, M., Ivanova, E., Kelsey, G., Bongcam-Rudloff, E., Andersson, G. and Humblot, P. (2020). LPS-treatment of bovine endometrial epithelial cells causes differential DNA methylation of genes associated with inflammation and endometrial function. <em>BMC Genomics</em>, <em>21</em>(1), 385. p. <a href="https://doi.org/10.1186/s12864-020-06777-7">https://doi.org/10.1186/s12864-020-06777-7</a>
</div></li>
<li><div id="ref-kumar2020genome" class="csl-entry" role="listitem">
Kumar, V., Donev, E. N., Barbut, F. R., Kushwah, S., Mannapperuma, C., Urbancsok, J. and Mellerowicz, E. J. (2020). Genome-wide identification of populus malectin/malectin-like domain-containing proteins and expression analyses reveal novel candidates for signaling and regulation of wood development. <em>Frontiers in Plant Science</em>, <em>11</em>, 588846. p. <a href="https://doi.org/10.3389/fpls.2020.588846">https://doi.org/10.3389/fpls.2020.588846</a>
</div></li>
<li><div id="ref-lakehal2020ethylene" class="csl-entry" role="listitem">
Lakehal, A., Dob, A., Rahneshan, Z., Novák, O., Escamez, S., Alallaq, S., Strnad, M., Tuominen, H. and Bellini, C. (2020). ETHYLENE RESPONSE FACTOR 115 integrates jasmonate and cytokinin signaling machineries to repress adventitious rooting in arabidopsis. <em>New Phytologist</em>, <em>228</em>(5), 1611–1626. pp. <a href="https://doi.org/10.1111/nph.16794">https://doi.org/10.1111/nph.16794</a>
</div></li>
<li><div id="ref-lecoy2020evolutionary" class="csl-entry" role="listitem">
Lecoy, J. and Garciá-Gil, M. (2020). Evolutionary analysis of LP3 gene family in conifers: An ASR homolog. <em>bioRxiv</em>, 2020–2003. pp. <a href="https://doi.org/10.1101/2020.03.27.011197">https://doi.org/10.1101/2020.03.27.011197</a>
</div></li>
<li><div id="ref-mannapperuma2020plantgenie" class="csl-entry" role="listitem">
Mannapperuma, C., Liu, H., Bel, M., Delhomme, N., Serrano, A., Schiffthaler, B., Hvidsten, T. R., Vandepoele, K., Ayllón-Benı́tez, A. and Street, N. (2020). Plantgenie-plaza: Integrating orthology into the plantgenie. Org resource using the plaza pipeline [Preprint]. <em>Umeå Universitet, Diva Portal</em>.
</div></li>
<li><div id="ref-martin2020transcriptional" class="csl-entry" role="listitem">
Martin, F., Dube, F., Karlsson Lindsjö, O., Eydal, M., Höglund, J., Bergström, T. F. and Tydén, E. (2020). Transcriptional responses in parascaris univalens after in vitro exposure to ivermectin, pyrantel citrate and thiabendazole. <em>Parasites &amp; Vectors</em>, <em>13</em>(1), 342. p. <a href="https://doi.org/0.1186/s13071-020-04212-0">https://doi.org/0.1186/s13071-020-04212-0</a>
</div></li>
<li><div id="ref-moestedt2020acetate" class="csl-entry" role="listitem">
Moestedt, J., Müller, B., Nagavara Nagaraj, Y. and Schnürer, A. (2020). Acetate and lactate production during two-stage anaerobic digestion of food waste driven by lactobacillus and aeriscardovia. <em>Frontiers in Energy Research</em>, <em>8</em>, 105. p. <a href="https://doi.org/10.3389/fenrg.2020.00105">https://doi.org/10.3389/fenrg.2020.00105</a>
</div></li>
<li><div id="ref-bioengineering7010003" class="csl-entry" role="listitem">
Moestedt, J., Westerholm, M., Isaksson, S. and Schnürer, A. (2020). Inoculum source determines acetate and lactate production during anaerobic digestion of sewage sludge and food waste. <em>Bioengineering</em>, <em>7</em>(1). <a href="https://doi.org/10.3390/bioengineering7010003">https://doi.org/10.3390/bioengineering7010003</a>
</div></li>
<li><div id="ref-obi2020stabilization" class="csl-entry" role="listitem">
Obi, I., Rentoft, M., Singh, V., Jamroskovic, J., Chand, K., Chorell, E., Westerlund, F. and Sabouri, N. (2020). Stabilization of g-quadruplex DNA structures in schizosaccharomyces pombe causes single-strand DNA lesions and impedes DNA replication. <em>Nucleic Acids Research</em>, <em>48</em>(19), 10998–11015. pp. <a href="https://doi.org/10.1093/nar/gkaa820">https://doi.org/10.1093/nar/gkaa820</a>
</div></li>
<li><div id="ref-WOS:000492817100001" class="csl-entry" role="listitem">
Peura, S., Wauthy, M., Simone, D., Eiler, A., Einarsdottir, K., Rautio, M. and Bertilsson, S. (2020). Ontogenic succession of thermokarst thaw ponds is linked to dissolved organic matter quality and microbial degradation potential [Article]. <em>LIMNOLOGY AND OCEANOGRAPHY</em>, <em>65</em>(1, SI), S248–S263. pp. <a href="https://doi.org/10.1002/lno.11349">https://doi.org/10.1002/lno.11349</a>
</div></li>
<li><div id="ref-Peura2020_ThermokarstSuccession" class="csl-entry" role="listitem">
Peura, S., Wauthy, M., Simone, D., Eiler, A., Einarsdóttir, K., Rautio, M. and Bertilsson, S. (2020). Ontogenic succession of thermokarst thaw ponds is linked to dissolved organic matter quality and microbial degradation potential. <em>Limnology and Oceanography</em>, <em>65</em>(S1), S248–S263. pp. <a href="https://doi.org/10.1002/lno.11349">https://doi.org/10.1002/lno.11349</a>
</div></li>
<li><div id="ref-senanayake2020genome" class="csl-entry" role="listitem">
Senanayake, K. S., Söderberg, J., Polajev, A., Malmberg, M., Karunanayake, E. H., Tennekoon, K. H., Samarakoon, S. R., Bongcam-Rudloff, E. and Niazi, A. (2020). The genome of setaria digitata: A cattle nematode closely related to human filarial parasites. <em>Genome Biology and Evolution</em>, <em>12</em>(2), 3971–3976. pp. <a href="https://doi.org/10.1093/gbe/evaa017 ">https://doi.org/10.1093/gbe/evaa017 </a>
</div></li>
<li><div id="ref-sullivan2020mitogenome" class="csl-entry" role="listitem">
Sullivan, A. R., Eldfjell, Y., Schiffthaler, B., Delhomme, N., Asp, T., Hebelstrup, K. H., Keech, O., Öberg, L., Møller, I. M., Arvestad, L., et al. (2020). The mitogenome of norway spruce and a reappraisal of mitochondrial recombination in plants. <em>Genome Biology and Evolution</em>, <em>12</em>(1), 3586–3598. pp. <a href="https://doi.org/10.1093/gbe/evz263">https://doi.org/10.1093/gbe/evz263</a>
</div></li>
<li><div id="ref-WILLE2020104290" class="csl-entry" role="listitem">
Wille, M., Wensman, J. J., Larsson, S., van Damme, R., Theelke, A.-K., Hayer, J. and Malmberg, M. (2020). Evolutionary genetics of canine respiratory coronavirus and recent introduction into swedish dogs. <em>Infection, Genetics and Evolution</em>, <em>82</em>, 104290. p. https://doi.org/<a href="https://doi.org/10.1016/j.meegid.2020.104290">https://doi.org/10.1016/j.meegid.2020.104290</a>
</div></li>
<li><div id="ref-zhang2020chromatin" class="csl-entry" role="listitem">
Zhang, B., Sztojka, B., Seyfferth, C., Escamez, S., Miskolczi, P., Chantreau, M., Bakó, L., Delhomme, N., Gorzsás, A., Bhalerao, R. P., et al. (2020). The chromatin-modifying protein HUB2 is involved in the regulation of lignin composition in xylem vessels. <em>Journal of Experimental Botany</em>, <em>71</em>(18), 5484–5494. pp. <a href="https://doi.org/10.1093/jxb/eraa264">https://doi.org/10.1093/jxb/eraa264</a>
</div></li>
</ol>
</section>
<section class="level2">
<h2 class="anchored" data-anchor-id="">2019</h2>
<ol start="153" type="1">
<li><div id="ref-bai2019transcriptome" class="csl-entry" role="listitem">
Bai, B., Horst, S. van der, Delhomme, N., Robles, A. V., Bentsink, L. and Hanson, J. (2019). Transcriptome and translatome profiling and translational network analysis during seed maturation reveals conserved transcriptional and distinct translational regulatory patterns. <em>bioRxiv</em>, 778001. p. <a href="https://doi.org/10.1101/778001">https://doi.org/10.1101/778001</a>
</div></li>
<li><div id="ref-blokhina2019ray" class="csl-entry" role="listitem">
Blokhina, O., Laitinen, T., Hatakeyama, Y., Delhomme, N., Paasela, T., Zhao, L., Street, N. R., Wada, H., Kärkönen, A. and Fagerstedt, K. (2019). Ray parenchymal cells contribute to lignification of tracheids in developing xylem of norway spruce. <em>Plant Physiology</em>, <em>181</em>(4), 1552–1572. pp. <a href="https://doi.org/10.1104/pp.19.00743">https://doi.org/10.1104/pp.19.00743</a>
</div></li>
<li><div id="ref-collani2019modulates" class="csl-entry" role="listitem">
Collani, S., Neumann, M., Yant, L. and Schmid, M. (2019). FT modulates genome-wide DNA-binding of the bZIP transcription factor FD. <em>Plant Physiology</em>, <em>180</em>(1), 367–380. pp. <a href="https://doi.org/10.1104/pp.18.01505">https://doi.org/10.1104/pp.18.01505</a>
</div></li>
<li><div id="ref-WOS:000471238300074" class="csl-entry" role="listitem">
Dahlberg, J., Sun, L., Waller, K. P., Ostensson, K., McGuire, M., Agenas, S. and Dicksved, J. (2019). Microbiota data from low biomass milk samples is markedly affected by laboratory and reagent contamination [Article]. <em>PLoS One</em>, <em>14</em>(6). <a href="https://doi.org/10.1371/journal.pone.0218257">https://doi.org/10.1371/journal.pone.0218257</a>
</div></li>
<li><div id="ref-WOS:000671007800010" class="csl-entry" role="listitem">
Giacomello, S., Delhomme, N., Niittyla, T., Tuominen, H. and Street, N. R. (2019). High spatial resolution profiling in tree species [Article]. <em>Annual Plant Reviews Online</em>, <em>2</em>(1), 329–359. pp. <a href="https://doi.org/10.1002/9781119312994.apr0688">https://doi.org/10.1002/9781119312994.apr0688</a>
</div></li>
<li><div id="ref-WOS:000459315700021" class="csl-entry" role="listitem">
Gourle, H., Karlsson-Lindsjo, O., Hayer, J. and Bongcam-Rudloff, E. (2019). Simulating illumina metagenomic data with InSilicoSeq [Article]. <em>Bioinformatics</em>, <em>35</em>(3), 521–522. pp. <a href="https://doi.org/10.1093/bioinformatics/bty630">https://doi.org/10.1093/bioinformatics/bty630</a>
</div></li>
<li><div id="ref-WOS:000532084200031" class="csl-entry" role="listitem">
Guo, Y., Schaik, T. van, Jhamat, N., Niazi, A., Chanrot, M., Charpigny, G., Valarcher, J. F., Bongcam-Rudloff, E., Andersson, G. and Humblot, P. (2019). Differential gene expression in bovine endometrial epithelial cells after challenge with LPS; specific implications for genes involved in embryo maternal interactions [Article]. <em>PLoS One</em>, <em>14</em>(9). <a href="https://doi.org/10.1371/journal.pone.0222081">https://doi.org/10.1371/journal.pone.0222081</a>
</div></li>
<li><div id="ref-WOS:000501767700025" class="csl-entry" role="listitem">
Kloth, K. J., Abreu, I. N., Delhomme, N., Petrik, I., Villard, C., Strom, C., Amini, F., Novak, O., Moritz, T. and Albrectsen, B. R. (2019). PECTIN ACETYLESTERASE9 affects the transcriptome and metabolome and delays aphid Feeding1<span>[</span>OPEN] [Article]. <em>Plant Physiology</em>, <em>181</em>(4), 1704–1720. pp. <a href="https://doi.org/10.1104/pp.19.00635">https://doi.org/10.1104/pp.19.00635</a>
</div></li>
<li><div id="ref-kumar2019poplar" class="csl-entry" role="listitem">
Kumar, V., Hainaut, M., Delhomme, N., Mannapperuma, C., Immerzeel, P., Street, N. R., Henrissat, B. and Mellerowicz, E. J. (2019). Poplar carbohydrate-active enzymes: Whole-genome annotation and functional analyses based on RNA expression data [Article]. <em>Plant Journal</em>, <em>99</em>(4), 589–609. pp. <a href="https://doi.org/10.1111/tpj.14417">https://doi.org/10.1111/tpj.14417</a>
</div></li>
<li><div id="ref-WOS:000481617000077" class="csl-entry" role="listitem">
Lopez-Fernandez, M., Broman, E., Simone, D., Bertilsson, S. and Dopson, M. (2019). Statistical analysis of community RNA transcripts between organic carbon and geogas-fed continental deep biosphere groundwaters [Article]. <em>MBIO</em>, <em>10</em>(4). <a href="https://doi.org/10.1128/mBio.01470-19">https://doi.org/10.1128/mBio.01470-19</a>
</div></li>
<li><div id="ref-mannapperuma2019designing" class="csl-entry" role="listitem">
Mannapperuma, C., Street, N. and Waterworth, J. (2019). Designing usable bioinformatics tools for specialized users. <em>International Conference on Information Technology &amp; Systems</em>, 649–670. pp. <a href="https://doi.org/10.1007/978-3-030-11890-7_62">https://doi.org/10.1007/978-3-030-11890-7_62</a>
</div></li>
<li><div id="ref-mannapperuma2019genie" class="csl-entry" role="listitem">
Mannapperuma, C., Waterworth, J. and Street, N. (2019). GenIE-sys: Genome integrative explorer system. <em>bioRxiv</em>, 808881. p. <a href="https://doi.org/10.1101/808881">https://doi.org/10.1101/808881</a>
</div></li>
<li><div id="ref-WOS:000587358700001" class="csl-entry" role="listitem">
Odilbekov, F., Armoniene, R., Koc, A., Svensson, J. and Chawade, A. (2019). GWAS-assisted genomic prediction to predict resistance to septoria tritici blotch in nordic winter wheat at seedling stage [Article]. <em>FRONTIERS IN GENETICS</em>, <em>10</em>. <a href="https://doi.org/10.3389/fgene.2019.01224">https://doi.org/10.3389/fgene.2019.01224</a>
</div></li>
<li><div id="ref-WOS:000487728800071" class="csl-entry" role="listitem">
Odilbekov, F., He, X., Armoniene, R., Saripella, G. V., Henriksson, T., Singh, P. K. and Chawade, A. (2019). QTL mapping and transcriptome analysis to identify differentially expressed genes induced by septoria tritici blotch disease of wheat [Article]. <em>Agronomy-Basel</em>, <em>9</em>(9). <a href="https://doi.org/10.3390/agronomy9090510">https://doi.org/10.3390/agronomy9090510</a>
</div></li>
<li><div id="ref-WOS:000502292300055" class="csl-entry" role="listitem">
Ohlund, P., Hayer, J., Lunden, H., Hesson, J. C. and Blomstrom, A.-L. (2019). Viromics reveal a number of novel RNA viruses in swedish mosquitoes [Article]. <em>Viruses</em>, <em>11</em>(11). <a href="https://doi.org/10.3390/v11111027">https://doi.org/10.3390/v11111027</a>
</div></li>
<li><div id="ref-WOS:000484936300056" class="csl-entry" role="listitem">
Ranade, S. S., Delhomme, N. and Garcia-Gil, M. R. (2019a). Global gene expression analysis in etiolated and de-etiolated seedlings in conifers [Article]. <em>PLoS One</em>, <em>14</em>(7). <a href="https://doi.org/10.1371/journal.pone.0219272">https://doi.org/10.1371/journal.pone.0219272</a>
</div></li>
<li><div id="ref-WOS:000471058300021" class="csl-entry" role="listitem">
Ranade, S. S., Delhomme, N. and Garcia-Gil, M. R. (2019b). Transcriptome analysis of shade avoidance and shade tolerance in conifers [Article]. <em>PLANTA</em>, <em>250</em>(1), 299–318. pp. <a href="https://doi.org/10.1007/s00425-019-03160-z">https://doi.org/10.1007/s00425-019-03160-z</a>
</div></li>
<li><div id="ref-schiffthaler2019seidr" class="csl-entry" role="listitem">
Schiffthaler, B., Serrano, A., Street, N. and Delhomme, N. (2019). <a href="">Seiðr: A gene meta-network calculation toolkit</a>. <em>bioRxiv</em>, <em>4</em>, 250696. p.
</div></li>
<li><div id="ref-WOS:000487981600001" class="csl-entry" role="listitem">
Seyfferth, C., Wessels, B. A., Gorzsas, A., Love, J. W., Ruggeberg, M., Delhomme, N., Vain, T., Antos, K., Tuominen, H., Sundberg, B. and Felten, J. (2019). Ethylene signaling is required for fully functional tension wood in hybrid aspen [Article]. <em>Frontiers in Plant Science</em>, <em>10</em>. <a href="https://doi.org/10.3389/fpls.2019.01101">https://doi.org/10.3389/fpls.2019.01101</a>
</div></li>
<li><div id="ref-singh2019acetobase" class="csl-entry" role="listitem">
Singh, A., Mueller, B., Fuxelius, H.-H. and Schnurer, A. (2019). AcetoBase: A functional gene repository and database for formyltetrahydrofolate synthetase sequences [Article]. <em>Database</em>. <a href="https://doi.org/10.1093/database/baz142">https://doi.org/10.1093/database/baz142</a>
</div></li>
<li><div id="ref-vain2019selective" class="csl-entry" role="listitem">
Vain, T., Raggi, S., Ferro, N., Barange, D. K., Kieffer, M., Ma, Q., Doyle, S. M., Thelander, M., Parizkova, B., Novak, O., Ismail, A., Enquist, P.-A., Rigal, A., Langowska, M., Harborough, S. R., Zhang, Y., Ljung, K., Callis, J., Almqvist, F., … Robert, S. (2019). Selective auxin agonists induce specific AUX/IAA protein degradation to modulate plant development [Article]. <em>Proceedings of the National Academy of Sciences</em>, <em>116</em>(13), 6463–6472. pp. <a href="https://doi.org/10.1073/pnas.1809037116">https://doi.org/10.1073/pnas.1809037116</a>
</div></li>
<li><div id="ref-WOS:000474107100001" class="csl-entry" role="listitem">
Wessels, B., Seyfferth, C., Escamez, S., Vain, T., Antos, K., Vahala, J., Delhomme, N., Kangasjarvi, J., Eder, M., Felten, J. and Tuominen, H. (2019). An AP2/ERF transcription factor ERF139 coordinates xylem cell expansion and secondary cell wall deposition [Article]. <em>New Phytologist</em>, <em>224</em>(4), 1585–1599. pp. <a href="https://doi.org/10.1111/nph.15960">https://doi.org/10.1111/nph.15960</a>
</div></li>
</ol>
</section>
<section class="level2">
<h2 class="anchored" data-anchor-id="">2018</h2>
<ol start="175" type="1">
<li><div id="ref-WOS:000449948700001" class="csl-entry" role="listitem">
Akhter, S., Kretzschmar, W. W., Nordal, V., Delhomme, N., Street, N. R., Nilsson, O., Emanuelsson, O. and Sundstrom, J. F. (2018). Integrative analysis of three RNA sequencing methods identifies mutually exclusive exons of MADS-box isoforms during early bud development in picea abies [Article]. <em>Frontiers in Plant Science</em>, <em>9</em>. <a href="https://doi.org/10.3389/fpls.2018.01625">https://doi.org/10.3389/fpls.2018.01625</a>
</div></li>
<li><div id="ref-WOS:000443861300009" class="csl-entry" role="listitem">
Capovilla, G., Delhomme, N., Collani, S., Shutava, I., Bezrukov, I., Symeonidi, E., Amorim, M. de F., Laubinger, S. and Schmid, M. (2018). PORCUPINE regulates development in response to temperature through alternative splicing [Article]. <em>Nature Plants</em>, <em>4</em>(8), 534–539. pp. <a href="https://doi.org/10.1038/s41477-018-0176-z">https://doi.org/10.1038/s41477-018-0176-z</a>
</div></li>
<li><div id="ref-WOS:000430418600001" class="csl-entry" role="listitem">
Cholleti, H., Hayer, J., Fafetine, J., Berg, M. and Blomstrom, A.-L. (2018). Genetic characterization of a novel picorna-like virus in &amp;ITCulex&amp;IT spp. Mosquitoes from mozambique [Article]. <em>Virology Journal</em>, <em>15</em>. <a href="https://doi.org/10.1186/s12985-018-0981-z">https://doi.org/10.1186/s12985-018-0981-z</a>
</div></li>
<li><div id="ref-collani2018effects" class="csl-entry" role="listitem">
Collani, S., Neumann, M., Yant, L. and Schmid, M. (2018). Effects of FLOWERING LOCUS t on FD during the transition to flowering at the shoot apical meristem of arabidopsis thaliana. <em>bioRxiv</em>, 483925. p. <a href="https://doi.org/10.1101/483925">https://doi.org/10.1101/483925</a>
</div></li>
<li><div id="ref-WOS:000451529600024" class="csl-entry" role="listitem">
Delhomme, F., Pallud, B. and Rouane, N. (2018). Tightening torque influence on pullout behavior of post-installed expansion anchors [Article]. <em>KSCE Journal of Civil Engineering</em>, <em>22</em>(10), 3931–3939. pp. <a href="https://doi.org/10.1007/s12205-018-0930-9">https://doi.org/10.1007/s12205-018-0930-9</a>
</div></li>
<li><div id="ref-felten2018ethylene" class="csl-entry" role="listitem">
Felten, J., Vahala, J., Love, J., Gorzsas, A., Ruggeberg, M., Delhomme, N., Lesniewska, J., Kangasjarvi, J., Hvidsten, T. R., Mellerowicz, E. J. and Sundberg, B. (2018). Ethylene signaling induces gelatinous layers with typical features of tension wood in hybrid aspen [Article]. <em>New Phytologist</em>, <em>218</em>(3, SI), 999–1014. pp. <a href="https://doi.org/10.1111/nph.15078">https://doi.org/10.1111/nph.15078</a>
</div></li>
<li><div id="ref-ferro2018isolation" class="csl-entry" role="listitem">
Ferro, L., Gentili, F. G. and Funk, C. (2018). Isolation and characterization of microalgal strains for biomass production and wastewater reclamation in northern sweden [Article]. <em>Algal Research</em>, <em>32</em>, 44–53. pp. <a href="https://doi.org/10.1016/j.algal.2018.03.006">https://doi.org/10.1016/j.algal.2018.03.006</a>
</div></li>
<li><div id="ref-jenjaroenpun2018complete" class="csl-entry" role="listitem">
Jenjaroenpun, P., Wongsurawat, T., Pereira, R., Patumcharoenpol, P., Ussery, D. W., Nielsen, J. and Nookaew, I. (2018). Complete genomic and transcriptional landscape analysis using third-generation sequencing: A case study of saccharomyces cerevisiae CEN. PK113-7D. <em>Nucleic Acids Research</em>, <em>46</em>(7), e38–e38. pp. <a href="https://doi.org/10.1093/nar/gky014">https://doi.org/10.1093/nar/gky014</a>
</div></li>
<li><div id="ref-jokipii2018transcriptional" class="csl-entry" role="listitem">
Jokipii-Lukkari, S., Delhomme, N., Schiffthaler, B., Mannapperuma, C., Prestele, J., Nilsson, O., Street, N. R. and Tuominen, H. (2018). Transcriptional roadmap to seasonal variation in wood formation of norway spruce [Article]. <em>Plant Physiology</em>, <em>176</em>(4), 2851–2870. pp. <a href="https://doi.org/10.1104/pp.17.01590">https://doi.org/10.1104/pp.17.01590</a>
</div></li>
<li><div id="ref-l2018topoisomerase" class="csl-entry" role="listitem">
L Pourbozorgi, P., Delhomme, N., Obi, I., Mohammad, J. B. and Sabouri, N. (2018). Topoisomerase 1 and the Pfh1 helicase are both required for proper DNA synthesis [Preprint]. <em>Diva, Umeå University</em>.
</div></li>
<li><div id="ref-l2018top1" class="csl-entry" role="listitem">
L Pourbozorgi, P., Delhomme, N., Obi, I. and Sabouri, N. (2018). Top1 and Pfh1Pif1 maintain the s. Pombe mitochondrial genome [Preprint]. <em>Diva, Umeå University</em>.
</div></li>
<li><div id="ref-law2018darkened" class="csl-entry" role="listitem">
Law, S. R., Chrobok, D., Juvany, M., Delhomme, N., Linden, P., Brouwer, B., Ahad, A., Moritz, T., Jansson, S., Gardestrom, P. and Keech, O. (2018). Darkened leaves use different metabolic strategies for senescence and survival [Article]. <em>Plant Physiology</em>, <em>177</em>(1), 132–150. pp. <a href="https://doi.org/10.1104/pp.18.00062">https://doi.org/10.1104/pp.18.00062</a>
</div></li>
<li><div id="ref-WOS:000468819504344" class="csl-entry" role="listitem">
Leblay, N., Alcala, N., Marin, D. H., Delhomme, T. M., Giffon, T., Ghantous, A., Chabrier, A., Cuenin, C., Altmueller, J., Durand, G., Voegele, C., Lorimier, P., Toffart, A.-C., Derks, J., Brustugun, O. T., Clement, J. H., Saenger, J., Field, J. K., Soltermann, A., … Fernandez-Cuesta, L. (2018). Multi-omics comparative analyses of pulmonary typical carcinoids, atypical carcinoids, and large-cell neuroendocrine carcinoma [Meeting Abstract]. <em>Cancer Research</em>, <em>78</em>(13, S). <a href="https://doi.org/10.1158/1538-7445.AM2018-5358">https://doi.org/10.1158/1538-7445.AM2018-5358</a>
</div></li>
<li><div id="ref-WOS:000449934400020" class="csl-entry" role="listitem">
Lin, Y.-C., Wang, J., Delhomme, N., Schiffthaler, B., Sundstrom, G., Zuccolo, A., Nystedt, B., Hvidsten, T. R., Torre, A. de la, Cossu, R. M., Hoeppner, M. P., Lantz, H., Scofield, D. G., Zamani, N., Johansson, A., Mannapperuma, C., Robinson, K. M., Mahler, N., Leitch, I. J., … Street, N. R. (2018). Functional and evolutionary genomic inferences in populus through genome and population sequencing of american and european aspen [Article]. <em>Proceedings of the National Academy of Sciences of the United States of America</em>, <em>115</em>(46), E10970–E10978. pp. <a href="https://doi.org/10.1073/pnas.1801437115">https://doi.org/10.1073/pnas.1801437115</a>
</div></li>
<li><div id="ref-manzoor2018genome" class="csl-entry" role="listitem">
Manzoor, S., Schnurer, A., Bongcam-Rudloff, E. and Muller, B. (2018). Genome-guided analysis of clostridium ultunense and comparative genomics reveal different strategies for acetate oxidation and energy conservation in syntrophic acetate-oxidising bacteria [Article]. <em>Genes</em>, <em>9</em>(4). <a href="https://doi.org/10.3390/genes9040225">https://doi.org/10.3390/genes9040225</a>
</div></li>
<li><div id="ref-reza2018transcriptome" class="csl-entry" role="listitem">
Reza, S. H., Delhomme, N., Street, N. R., Ramachandran, P., Dalman, K., Nilsson, O., Minina, E. A. and Bozhkov, P. V. (2018). Transcriptome analysis of embryonic domains in norway spruce reveals potential regulators of suspensor cell death [Article]. <em>PLoS One</em>, <em>13</em>(3). <a href="https://doi.org/10.1371/journal.pone.0192945">https://doi.org/10.1371/journal.pone.0192945</a>
</div></li>
<li><div id="ref-seyfferth2018ethylene" class="csl-entry" role="listitem">
Seyfferth, C., Wessels, B., Jokipii-Lukkari, S., Sundberg, B., Delhomme, N., Felten, J. and Tuominen, H. (2018). Ethylene-related gene expression networks in wood formation [Article]. <em>Frontiers in Plant Science</em>, <em>9</em>. <a href="https://doi.org/10.3389/fpls.2018.00272">https://doi.org/10.3389/fpls.2018.00272</a>
</div></li>
<li><div id="ref-singh2018genetic" class="csl-entry" role="listitem">
Singh, R. K., Maurya, J. P., Azeez, A., Miskolczi, P., Tylewicz, S., Stojkovic, K., Delhomme, N., Busov, V. and Bhalerao, R. P. (2018). A genetic network mediating the control of bud break in hybrid aspen [Article]. <em>Nature Communications</em>, <em>9</em>. <a href="https://doi.org/10.1038/s41467-018-06696-y">https://doi.org/10.1038/s41467-018-06696-y</a>
</div></li>
<li><div id="ref-sundstrom2018reproducibility" class="csl-entry" role="listitem">
Sundström, G., Schneider, A. N., Richau, K., Haas, J. C., Delhomme, N., Sjödin, A., Hurry, V., Grabherr, M. and Street, N. R. (2018). Reproducibility and biological information of tree-associated metacommunities using RNA sequencing and DNA amplicon profiling. <em>Diva</em>. <a href="https://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1259386&amp;dswid=-7962">https://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1259386&amp;dswid=-7962</a>
</div></li>
<li><div id="ref-zare2018gut" class="csl-entry" role="listitem">
Zare, A., Johansson, A.-M., Karlsson, E., Delhomme, N. and Stenberg, P. (2018). The gut microbiome participates in transgenerational inheritance of low-temperature responses in drosophila melanogaster [Article]. <em>Febs Letters</em>, <em>592</em>(24), 4078–4086. pp. <a href="https://doi.org/10.1002/1873-3468.13278">https://doi.org/10.1002/1873-3468.13278</a>
</div></li>
</ol>
</section>
</div>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp("https:\/\/www\.slubi\.se");
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>